<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<base href="/" />
<title>FAQ &middot; TFlink</title>
<link rel="stylesheet" href="https://cdn.webix.com/edge/webix.css" />
<link
  rel="stylesheet"
  href="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/css/uikit.min.css"
/>


<link href="/main.41f68.css" rel="stylesheet" />

<script
  src="https://code.jquery.com/jquery-3.4.1.min.js"
  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
  crossorigin="anonymous"
></script>
<script src="https://cdn.webix.com/edge/webix.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit-icons.min.js"></script>

  </head>

  <body>
    <div class="net-top-navbar" uk-sticky>
    <div class="uk-container uk-container-center uk-container-expand">
        <nav class="uk-navbar uk-navbar-container navbar">
            <a class="uk-navbar-item uk-logo" href="/">TFlink</a>
            <ul class="uk-navbar-nav">
                <li>
                    <a href="/browse/">Browse</a>
                </li>
                <li>
                  <a href="/download/">Download</a>
              </li>
              <li>
                <a href="/faq/">FAQ</a>
            </li>
            </ul>
        </nav>
    </div>
</div>


    <div class="uk-container uk-container-expand uk-container-center uk-margin-large-bottom">
      
<div>
  <h2>Who worked on the <i>TFLink</i> project?</h2>
<p><strong>Orsolya Liska</strong></p>
<p>liska.orsolya@brc.mta.hu</p>
<p>webpage</p>
<p>Affiliations:</p>
<ul>
  <li>Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</li>
<li>Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary</li>
<li>Department of Genetics, Eötvös Loránd University, Budapest, Hungary</li>
</ul>

<p><strong>Balázs Bohár</strong></p>
<p>bbazsi41@gmail.com</p>
<p>Affiliations:</p>
<li>Department of Genetics, Eötvös Loránd University, Budapest, Hungary</li>
<li>Earlham Institute, Norwich, UK</li></ul>

<p><strong>András Hidas</strong></p>
<p>hidasandris@gmail.com</p>
<p>Affiliation:</p>
<ul><li>MTA Centre for Ecological Research, Danube Research Institute, Budapest, Hungary</li></ul></ul>

<p><strong>Tamás Korcsmáros</strong></p>
<p>Tamas.Korcsmaros@earlham.ac.uk</p>
<p>webpage</p>
<p>Affiliations:</p>
<ul><li>Earlham Institute, Norwich, UK</li>
<li>Quadram Institute, Norwich UK</li></ul>

<p><strong>Balázs Papp</strong></p>
<p>papp.balazs@brc.mta.hu</p>
<p>webpage</p>
<p>Affiliations: </p>
<li>Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</li>
<li>Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary</li></ul>

<p><strong>Dávid Fazekas</strong></p>
<p>fazekas@netbiol.elte.hu</p>
<p>webpage</p>
<p>Affiliations:</p>
<li>Department of Genetics, Eötvös Loránd University, Budapest, Hungary</li>
<li>Earlham Institute, Norwich, UK</li></ul>

<p><strong>Eszter Ari</strong></p>
<p>arieszter@gmail.com</p>
<p>webpage1</p>
<p>webpage2</p>
<p>Affiliations: </p>
<li>Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</li>
<li>Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary</li>
<li>Department of Genetics, Eötvös Loránd University, Budapest, Hungary</li></ul>
<h2>What does a protein page contain?</h2>
<p>When you select a transcription factor or a target gene protein from the Browsing table a protein page will be opened.</p>
<h3>What does a transcription factor protein page contain?</h3>
<p>Besides basic information and a link to the <i>Uniprot</i> page of the protein there is a table that contains information about the target genes of the transcription factor.</p>
<p>If there are known binding sites of the transcription factor you will find a table containing their genomic locations and a list containing their DNA sequences in fasta format below the table.</p>
<h3>What does a target gene protein page contain?</h3>
<p>Besides basic information and a link to the <i>Uniprot</i> page of the protein there is a table that contains information about the transcription factors of the target gene.</p>
<h3>What does a transcription factor and target gene protein page contain?</h3>
<p>Some proteins act like transcription factors and target genes simultaneously. In some cases a transcription factor can regulate its own gene too.</p>
<p>Besides basic information and a link to the <i>Uniprot</i> page of the protein there is a table that contains information about the target genes of the protein.</p>
<p>There is also a table that contains information about the transcription factors of the protein.</p>
<p>If there are known binding sites of the transcription factor you will find a table containing their genomic locations and a list containing their DNA sequences in <i>fasta</i> format below the table.</p>
<h2>What are the downloadable data formats?</h2>
<h3>What do interaction tables contain?</h3>
<p>Interaction tables are comma separated tables (<i>csv</i>) of transcription factor - target gene interactions that contain either interactions validated by small-scale experiments (<i>small-scale interaction table</i>) or large-scale experiments (<i>large-scale interaction table</i>), along with the following data: <i>Uniprot IDs</i>, protein names, <i>NCBI Gene IDs</i> of the transcription factors and the target genes; the names of the detection methods; the <i>Pubmed IDs</i> of the original publications; the name of the organism and the source database.</p>
<h3>What do <i>MITAB</i> tables contain?</h3>
<p>Interaction MITABs contain transcription factor - target gene interactions in HUPO-PSI MITAB 2.8 format.</p>
<h3>What is the <i>MITAB</i> format?</h3>
<p><i>MITAB 2.8</i> is a standardized format introduced by the <i>Human Proteome Organization</i> (<i>HUPO</i>) which captures binary molecular interactions using standardized content and vocabulary. The <i>MITAB</i> files are tab-delimited tables containing 46 columns. A header for the <i>MITAB</i> tables is available here. </p>
<p>For more information on the <i>HUPO-PSI</i>'s molecular interaction format see: <u>link</u></p>
<p>Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of the <i>HUPO-PSI</i> format for molecular interactions. <i>BMC Bioinformatics</i> <strong>19</strong>(1):134. <u>doi: 10.1186/s12859-018-2118-1</u>.</p>
<h3>What do binding site tables contain?</h3>
<p>Binding site tables are comma separated tables (<i>csv</i>) of binding site annotations that contain the unique internal <i>TFLink IDs</i> of binding sites; <i>Uniprot IDs</i> and protein names of the transcription factors; the names of the detection method; <i>Pubmed IDs</i> of the original publications; the organism and the source database; the version of the genome assembly; the name of the chromosome; the start and end coordinates of site; and the strand.</p>
<h3>What do binding site sequence files contain?</h3>
<p>Binding site sequence files are <i>fasta</i> files containing the DNA sequences of the transcription factor binding sites. The header of each sequence contains the unique internal <i>TFLink ID</i> of the binding site; the <i>Uniprot ID</i> and protein name of the transcription factor, the version of the genome assembly; the name of the chromosome; and the the start and end coordinates of site.</p>

<h2>What source databases provided the data of TFLink?</h2>
<p>We use the following abbreviations in the table:</p>
<p>Data based on small-scale experiments: SS</p>
<p>Data based on large-scale experiments: LS</p>
<p><i>Homo sapiens</i>: Hs</p>
<p><i>Mus musculus</i>: Mm</p>
<p><i>Rattus norvegicus</i>: Rn</p>
<p><i>Danio rerio</i>: Dr</p>
<p><i>Drosophila melanogaster</i>: Dm</p>
<p><i>Caenorhabditis elegans</i>: Ce</p>
<p><i>Saccharomyces cerevisiae</i>: Sc</p>

<p><strong>Nr. of data downloaded</strong></p><p><strong>Version</strong></p><p><strong>Downloading date</strong></p><p><strong>Type of data</strong></p><p><strong>Nr. of integrated data</strong></p><p><strong>Species</strong></p><p><strong>GTRD</strong></p><p>18.06</p><p>29/08/2018</p><p>LS interactions</p><p>5,341,065</p><p>Hs, Mm, Rn, Dr, Ce, Sc</p><p>LS binding sites</p><p>5,341,065</p><p><strong>HTRIdb </strong></p><p>no longer available</p><p>29/04/2017</p><p>SS interactions</p><p>1793</p><p>Hs</p><p>LS interactions</p><p>47,565</p><p><strong>JASPAR</strong></p><p>2016</p><p>16/08/2017</p><p>SS binding sites</p><p>7761</p><p>Hs, Mm, Rn, Dm, Ce</p><p>LS binding sites</p><p>1,126,598</p><p><strong>ORegAnno</strong></p><p>2016 (the website is out of order)</p><p>24/05/2017</p><p>SS interactions</p><p>1,762</p><p>Hs, Mm, Rn, Dm, Ce, Sc</p><p>LS interactions</p><p>197,630</p><p>SS binding sites</p><p>47,409</p><p>LS binding sites</p><p>705,275</p><p><strong>REDfly</strong></p><p>5.5</p><p>03/09/2018</p><p>SS interactions</p><p>1,100</p><p>Dm</p><p>SS binding sites</p><p>1,787</p><p><strong>ReMap</strong></p><p>2018 (v1.2)</p><p>16/07/2018</p><p>LS interactions</p><p>3,011,117</p><p>Hs</p><p>LS binding sites</p><p>3,011,117</p><p><strong>TRED</strong></p><p>obtained from RegNetwork 2017</p><p>06/08/2018</p><p>SS interactions</p><p>8,938</p><p>Hs, Mm</p><p><strong>TRRUST</strong></p><p>2</p><p>30/07/2018</p><p>SS interactions</p><p>16,188</p><p>Hs, Mm</p><p><strong>Yeastract</strong></p><p>-</p><p>29/06/2017</p><p>SS interactions</p><p>159,625</p><p>Sc</p>
<h2>Where can I find the the interaction and binding site data?</h2>
<h3>Where can I find the interactions and binding sites based on small-scale experiments?</h3>
<p>For data of small-scale experiments we provide options for browsing, searching and downloading data in different standardized formats (<i>MITAB</i>, <i>csv</i>, <i>gmt</i> and <i>fasta</i> files that include <i>Uniprot IDs</i>, protein names, <i>PubMed IDs</i>, etc.). </p>
<h3>Where can I find the interactions and binding sites based on large-scale experiments?</h3>
<p>Because of their quantity, the interactions and binding sites based on large-scale experiments are only available in forms of downloadable files, therefore cannot be browsed on the <i>TFLink</i> website. The downloadable files are accessible in standard formats: <i>MITAB</i>, <i>csv</i>, <i>gmt</i> and <i>fasta</i> files that include <i>Uniprot IDs</i>, protein names, <i>PubMed IDs</i>, etc.</p>
<h3>Where can I find the data of <i>Danio rerio</i>?</h3>
<p>From <i>Danio rerio</i> only large-scale data was available in the source databases, therefore you can find these interaction and binding site data under the <i>Downloads</i> menu.</p>
<h2>Is <i>TFLink</i> freely available?</h2>
<p><i>TFLink</i> is freely available for non-commercial use.</p>
</div>

    </div>

    
    <footer></footer>

    

    
    
    <script src="/main.41f68.js"></script>
    
  </body>
</html>
