<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<base href="/" />
<title>FAQ &middot; TFlink</title>
<link rel="stylesheet" href="https://cdn.webix.com/edge/webix.css" />
<link
  rel="stylesheet"
  href="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/css/uikit.min.css"
/>


<link href="/main.43160.css" rel="stylesheet" />

<script
  src="https://code.jquery.com/jquery-3.4.1.min.js"
  integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="
  crossorigin="anonymous"
></script>
<script src="https://cdn.webix.com/edge/webix.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit-icons.min.js"></script>

  </head>

  <body>
    <div class="net-top-navbar" uk-sticky>
    <div class="uk-container uk-container-center uk-container-expand">
        <nav class="uk-navbar uk-navbar-container navbar">
            <a class="uk-navbar-item uk-logo" href="/">TFlink</a>
            <ul class="uk-navbar-nav">
                <li>
                    <a href="/browse/">Browse</a>
                </li>
                <li>
                  <a href="/download/">Download</a>
              </li>
              <li>
                <a href="/faq/">FAQ</a>
            </li>
            </ul>
        </nav>
    </div>
</div>


    <div class="uk-container uk-container-expand uk-container-center uk-margin-large-bottom">
      
<div class="uk-grid" data-uk-grid-margin>
  

<h2 id="who-was-working-on-the-tflink-project">Who was working on the TFLink project?</h2>

<p>Orsolya Liska<br />
liska.orsolya@brc.mta.hu<br />
webpage<br />
Affiliations:<br />
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary<br />
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary<br />
Department of Genetics, Eötvös Loránd University, Budapest, Hungary<br />
Balázs Bohár<br />
bbazsi41@gmail.com<br />
Affiliations:<br />
Department of Genetics, Eötvös Loránd University, Budapest, Hungary<br />
Earlham Institute, Norwich, UK<br />
András Hidas<br />
hidasandris@gmail.com<br />
Affiliation:<br />
MTA Centre for Ecological Research, Danube Research Institute, Budapest, Hungary<br />
Tamás Korcsmáros<br />
Tamas.Korcsmaros@earlham.ac.uk<br />
webpage<br />
Affiliations:<br />
Earlham Institute, Norwich, UK<br />
Quadram Institute, Norwich UK<br />
Balázs Papp<br />
papp.barazs@brc.mta.hu<br />
webpage<br />
Affiliations:<br />
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary<br />
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary<br />
Dávid Fazekas<br />
fazekas@netbiol.elte.hu<br />
webpage<br />
Affiliations:<br />
Department of Genetics, Eötvös Loránd University, Budapest, Hungary<br />
Earlham Institute, Norwich, UK<br />
Eszter Ari<br />
arieszter@gmail.com<br />
webpage1<br />
webpage2<br />
Affiliations:<br />
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary<br />
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary<br />
Department of Genetics, Eötvös Loránd University, Budapest, Hungary<br />
What are the downloadable data formats?<br />
What do interaction tables contain?<br />
Interaction tables are comma separated tables (CSV) of transcription factor - target gene interactions that contains either experimentally validated interaction (experiment interaction table) or predicted interactions (predicted experiment interaction table), along with the following data: Uniprot IDs, NCBI Gene IDs and gene names of the transcription factors and the target genes; the names of the detection methods; the Pubmed IDs of the original publications; the name of the organism and the source database.<br />
What do MITAB tables contain?<br />
Interaction MITABs contain transcription factor - target gene interactions in HUPO-PSI MITAB 2.8 format.<br />
What is MITAB format?<br />
MITAB 2.8 is a standardized format introduced by the Human Proteome Organization (HUPO) which captures binary molecular interactions using standardized content and vocabulary. The MITAB files are tab-delimited tables containing 46 columns. A header for the MITAB tables is available here.<br />
For more information on the HUPO-PSI&rsquo;s molecular interaction format see: link<br />
Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of the HUPO-PSI format for molecular interactions. BMC Bioinformatics 19(1):134. doi: 10.1186/s12859-018-2118-1.<br />
What do binding site tables contain?<br />
Binding site tables are comma separated tables (CSV) of binding site annotations that contain the unique internal TFLink IDs of binding sites; Uniprot IDs and protein names of the transcription factors; the names of the detection method; Pubmed IDs of the original publications; the organism and the source database; the version of the genome assembly; the name of the chromosome; the start and end coordinates of site; and the strand.<br />
What do binding site sequence files contain?<br />
Binding site sequences are FASTA files containing the DNA sequences of the transcription factor binding sites. The header of each sequence contains the unique internal TFLink ID of the binding site; the Uniprot ID and protein name of the transcription factor, the version of the genome assembly; and the the start and end coordinates of site.<br />
What source databases provide the data of TFLink?<br />
We use the following abbreviations in the table:<br />
Data based on low-throughput experiments: LT<br />
Data based on high-throughput experiments: HT<br />
Homo sapiens: Hs<br />
Mus musculus: Mm<br />
Rattus norvegicus: Rn<br />
Drosophila melanogaster: Dm<br />
Caenorhabditis elegans: Ce<br />
Saccharomyces cerevisiae: Sc</p>

<p>Nr. of data downloaded</p>

<p>Version<br />
Downloading date<br />
Type of data<br />
Nr. of integrated data<br />
species</p>

<p>GTRD</p>

<p>interactions</p>

<p>binding sites</p>

<p>HTRIdb<br />
no longer available<br />
29/04/2017<br />
interactions</p>

<p>Hs<br />
JASPAR<br />
2016<br />
16/08/2017<br />
binding sites</p>

<p>Hs, Mm, Rn, Dm, Ce<br />
ORegAnno<br />
2016 (the website is out of order)<br />
24/05/2017<br />
interactions</p>

<p>Hs, Mm, Rn, Dm, Ce, Sc</p>

<p>binding sites</p>

<p>REDfly<br />
5.5<br />
03/09/2018<br />
interactions</p>

<p>Dm<br />
binding sites</p>

<p>ReMap</p>

<p>interactions</p>

<p>binding sites</p>

<p>TRED<br />
obtained from RegNetwork 2017<br />
06/08/2018<br />
interactions</p>

<p>Hs, Mm<br />
TRRUST<br />
2<br />
30/07/2018<br />
interactions</p>

<p>Hs, Mm</p>

<h2 id="yeastract">Yeastract</h2>

<p>29/06/2017<br />
interactions</p>

<p>Sc</p>

<p>What kind of experimental methods does TFLink contain?</p>

</div>

    </div>

    
    <footer></footer>

    

    
    
    <script src="/main.43160.js"></script>
    
  </body>
</html>
