<!DOCTYPE html>
<html>

<head>
  <meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<base href="/" />
<title>TFLink</title>
<link rel="stylesheet" href="https://cdn.webix.com/edge/webix.css" />

<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon.png?v=jwEjQXwyNy">
<link rel="icon" type="image/png" sizes="32x32" href="/images/favicon/favicon-32x32.png?v=jwEjQXwyNy">
<link rel="icon" type="image/png" sizes="16x16" href="/images/favicon/favicon-16x16.png?v=jwEjQXwyNy">
<link rel="manifest" href="/images/favicon/site.webmanifest?v=jwEjQXwyNy">
<link rel="mask-icon" href="/images/favicon/safari-pinned-tab.svg?v=jwEjQXwyNy" color="#70adb5">
<link rel="shortcut icon" href="/images/favicon/favicon.ico?v=jwEjQXwyNy">
<meta name="msapplication-TileColor" content="#00aba9">
<meta name="msapplication-config" content="/images/favicon/browserconfig.xml?v=jwEjQXwyNy">
<meta name="theme-color" content="#ffffff">



<link href="/main.e9931.css" rel="stylesheet" />

<script src="https://code.jquery.com/jquery-3.5.1.min.js"
  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
<script src="https://cdn.webix.com/edge/webix.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.1.5/js/uikit-icons.min.js"></script>

<script>
  $(window).on('load', function () {
    $('#spinner').fadeOut(300);
  });
</script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-5RNBLSCVTD"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-5RNBLSCVTD');
</script>

</head>

<body>
  <div class="net-top-navbar" uk-sticky>
    <div class="uk-container uk-container-center uk-container-expand">
        <nav class="uk-navbar uk-navbar-container navbar">
            <a class="uk-navbar-item uk-logo" href="/"><img src="/images/favicon/favicon-32x32.png">Link</a>
            <ul class="uk-navbar-nav">
                <li id="nav_browse">
                    <a href="/browse/">Browse</a>
                </li>
                <li id="nav_download">
                  <a href="/download/">Download</a>
              </li>
              <li id="nav_faq">
                <a href="/faq/">FAQ</a>
            </li>
            </ul>
        </nav>
    </div>
</div>


  <div class="uk-container uk-container-expand uk-margin-large-bottom">
    <div id="spinner"></div>
    





<div class="uk-container uk-container-small">
  <span style="text-align: center">
    <h1>FAQ</h1>
  </span>
  <hr />

  <i href="javascript:" id="return-to-top"><span uk-icon="icon: chevron-up; ratio: 2"></span></i>

  <a href="faq/#faq_who-worked" uk-scroll>Who is working on the <i>TFLink</i> project?</a>
  <br />
  <a href="faq/#faq_how-to-browse" uk-scroll>How to browse the TFLink database?</a>
  <br />
  <a href="faq/#faq_what-does-an-entry-page-contain" uk-scroll>What does an entry page contain?</a>
  <br />
  <a href="faq/#faq_what-are-the-downloadable-data-formats" uk-scroll>What are the downloadable data
    formats?</a>
  <div class="margin-left">
    <a href="faq/#faq_what-do-interaction-tables-contain" uk-scroll>What do interaction tables
      contain?</a>
    <br />
    <a href="faq/#faq_what-do-mitab" uk-scroll>What do <i>MITAB</i> tables contain?</a>
    <br />
    <a href="faq/#faq_what-do-interaction-gmt" uk-scroll>What do interaction GMT files contain?</a>
    <br />
    <a href="faq/#faq_what-do-binding-site-tables-contain" uk-scroll>What do binding site tables
      contain?</a>
    <br />
    <a href="faq/#faq_what-do-gff3-binding-site-annotation-contain" uk-scroll>What do <i>GFF3</i>
      binding site annotation files contain?</a>
    <br />
    <a href="faq/#faq_what-do-binding-site-sequence-files-contain" uk-scroll>What do binding site
      sequence files contain?</a>
  </div>
  <a href="faq/#faq_what-source-databases" uk-scroll>What were the source databases that provided
    the data of <i>TFLink</i>?</a>
  <br />
  <a href="faq/#faq_what-kind-of-biologiacal-questions" uk-scroll>What kind of biological questions
    can be answered by using data on <i>TFLink?</i></a>
  <br />
  <a href="faq/#faq_is-tflink-freely-available" uk-scroll>Is <i>TFLink</i> freely available?</a>


  <a class="anchor" id="faq_who-worked"></a>
  <h3>Who is working on the <i>TFLink</i> project?</h3>

  <div class="uk-card uk-card-default">
    <strong>Orsolya Liska</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:liska.orsolya@brc.hu">liska.orsolya@brc.hu</a><br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#orsolya-liska-sarkadi"
      target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Bohár</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:bbazsi41@gmail.com">bbazsi41@gmail.com</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>András Hidas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:hidas.andras@ecolres.hu">hidas.andras@ecolres.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://ecolres.hu/en/Andras.HIDAS" target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://voi.ecolres.hu/en/node/14728" target="_blank">
          Institute of Aquatic Ecology, Centre for Ecological Research, Budapest,
          Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Tamás Korcsmáros</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:Tamas.Korcsmaros@earlham.ac.uk">Tamas.Korcsmaros@earlham.ac.uk</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://www.earlham.ac.uk/korcsmaros-group" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
      <li>
        <a href="https://quadram.ac.uk/" target="_blank">
          Quadram Institute, Norwich UK
        </a>
      </li>
      <li>
        <a href="https://www.imperial.ac.uk/people/t.korcsmaros" target="_blank">
          Imperial College London, Faculty of Medicine, London, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Papp</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:papp.balazs@brc.hu">papp.balazs@brc.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html"
      target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Dávid Fazekas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:fazekas@netbiol.elte.hu">fazekas@netbiol.elte.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://www.earlham.ac.uk/david-fazekas" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Eszter Ari</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148" target="_blank">website 1</a>,
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari"
      target="_blank">website 2</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <a class="anchor" id="faq_how-to-browse"></a>
  <h3>
    How to browse the TFLink database?
  </h3>
  <p class="justified">
    After selecting the organism, the user can browse and search within the dataset. The results can
    be filtered by <i>gene name, UniProt ID, NCBI Gene ID, function</i> (<i>e.g.</i> 'transcription
    factor',
    'target gene', or 'transcription factor and target gene'), and according to <i>evidence type</i>
    (small- or large-scale experiments). TFLink differentiates between <i>'transcription factor'</i>
    and
    <i>'transcription factor and target gene'</i> functions based on whether the transcription
    factor
    protein regulating the gene for the particular transcription factor is known (present in the
    TFLink database) or not. We suggest filtering for both 'transcription factor' and
    'transcription factor and target gene', and then focusing on the "Targets of …" table on the
    Entry site, when looking for transcription factors. Such as filtering 'target gene' and
    'transcription factor and target gene', and then focusing on the "Transcription factors of …"
    table on the Entry site, when searching target genes. Information on the number of interactions
    a particular gene or protein
    is involved in is also provided. After selecting an entry (gene or protein) from the Browsing
    table, an <i>'entry page'</i> is opened. (Links to example entry pages are provided in the
    Supplementary Notes and in the FAQ part of the TFLink gateway.)
  </p>

  <a class="anchor" id="faq_what-does-an-entry-page-contain"></a>
  <h3>
    What does an entry page contain?
  </h3>
  <div class="justified">
    <p>
      Each entry page contains basic information about the transcription factor protein or target
      gene: <i>gene name, UniProt ID</i> (linked to the corresponding <a
        href="https://www.uniprot.org/" target="_blank">UniProt</a> protein page), <i>NCBI Gene
        ID</i>
      (linked to the corresponding <a href="https://www.ncbi.nlm.nih.gov/gene" target="_blank">NCBI
        Gene</a> site), <i>organism</i> (the scientific name of the species), its
      <i>function</i> (transcription factor, target gene or both), the <i>number of its
        interactions</i>, and its
      <i>orthologs</i> (species name and <i>UniProt ID</i>) – when there are any. In case the
      ortholog
      is also
      available in the TFLink database, a link is provided to the related entry page. Binding site
      nucleotide composition f<i>requency matrices</i> and <i>sequence logos</i> of the
      transcription
      factors are
      also available through the <a href="https://jaspar.genereg.net/" target="_blank">JASPAR</a>
      website to facilitate the prediction of more binding sites.
    </p>
    <p>
      Below the basic information section, the user may visualise three layers of information (if
      available) about the selected transcription factor: (1) <i>target genes</i> of the
      transcription
      factor
      and/or (2) <i>transcription factors</i> for the target gene and (3) <i>binding sites</i> of
      the
      transcription
      factor. In the target gene and transcription factor tables the user finds details on gene
      names
      (linked to corresponding TFLink entries), <i>UniProt IDs</i> (linked to corresponding UniProt
      protein
      pages), <i>NCBI Gene IDs</i> (linked to corresponding NCBI Gene sites), name of the source
      database(s),
      method(s) of detection, cross-links to the original publications at <a
        href="https://pubmed.ncbi.nlm.nih.gov/" target="_blank">NCBI PubMed</a>, and
      indications
      of the evidence type (small- or large-scale experiments). Along with these tables, interactive
      network visualisations are presented, demonstrating the interactions between the transcription
      factor(s) and target gene(s) (indicated by green and red colours, respectively) to facilitate
      the visual inspection of the interactions.
    </p>
    <p>
      Besides the <i>TFLink ID</i>, the name of the source database(s), the method(s) of detection,
      the
      link
      to the original publications, and the indication to clarify whether the evidence is based on a
      small- or a large-scale experiment, the <i>binding site table</i> also presents information
      about the
      genomic location: genome assembly version, chromosome, the coordinates of the start and end
      points of transcription factor binding sites, and the number of overlapping binding sites for
      the particular transcription factor. To make the visual exploration of the genomic context
      easier, each binding site is linked to its particular genomic location at the <a
        href="https://genome.ucsc.edu/" target="_blank">UCSC genome browser</a> website.
    </p>
    <p>
      In case there are more than 100 interactions or binding sites available for a particular entry
      in the TFLink gateway, we only show the first 100 targets / transcription factors / binding
      sites in the tables on the website, and make the full information available in the form of
      <i>downloadable table</i> (and in case of binding sites: <a
        href="faq/#faq_what-do-gff3-binding-site-annotation-contain"><i>GFF3 annotation</i></a>)
      files.
    </p>
    <p>
      The sequences of binding sites based on small-scale evidence are shown below the tables in
      <i>FASTA</i>
      format. The header of the sequences contains the <i>TFLink</i> and the <i>UniProt IDs</i>,
      gene name, genome
      assembly version, chromosome name and the start and end point coordinates of the binding
      sites.
      Some data downloaded from the JASPAR database refer to binding sequences without exact
      localization, for example in cases when random sequences were investigated with SELEX. The
      binding sequences revealed by large-scale experiments are available from the entry pages as
      downloadable <a href="faq/#faq_what-do-binding-site-sequence-files-contain"><i>FASTA
          files</i></a>.
    </p>
    <p>
      <strong>Entry pages example links</strong>
    <ul>
      <li>Transcription factor entry page: <a href="https://tflink.net/protein/q9vhm6/"
          target="_blank">https://tflink.net/protein/q9vhm6/</a></li>
      <li>Target gene protein page: <a href="https://tflink.net/protein/p52429/"
          target="_blank">https://tflink.net/protein/p52429/</a></li>
      <li>Transcription factor and target gene protein page: <a
          href="https://tflink.net/protein/p10242/"
          target="_blank">https://tflink.net/protein/p10242/</a></li>
    </ul>
    </p>
  </div>

  <a class=" anchor" id="faq_what-are-the-downloadable-data-formats"></a>
  <h3>
    What are the downloadable data formats?
  </h3>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-interaction-tables-contain"></a>
    <h4>
      What do interaction tables contain?
    </h4>
    <p class="justified">
      Interaction table files are tab separated tables (<i>TSV</i>) of transcription factor - target
      gene
      interactions that contain either interactions validated by small-scale experiments or
      large-scale experiments or these two data altogether. Interaction tables contain the following
      data:
    <ol>
      <li><strong>UniprotID.TF</strong> and/or* <strong>UniprotID.Target:</strong>
        <i>Uniprot IDs</i> of
        transcription factors and/or target genes
      </li>
      <li>
        <strong>NCBI.GeneID.TF</strong> and/or <strong>NCBI.GeneID.Target:</strong> <i>NCBI Gene
          IDs</i> of
        transcription factors and/or target genes,
      </li>
      <li><strong>Name.TF</strong> and/or <strong>Name.Target:</strong> gene names of transcription
        factors and/or target genes,</li>
      <li><strong>Detection.method:</strong> names of the detection methods,</li>
      <li><strong>PubmedID:</strong> <i>Pubmed IDs</i> of the original publications (when available)
        and the publications of the databases,</li>
      <li><strong>Organism:</strong> scientific name of the organism,</li>
      <li><strong>Source.database:</strong> names of the original source databases, and</li>
      <li>
        <strong>Small-scale.evidence:</strong> indication about if the data were confirmed by
        small-scale evidence (with "Yes" or "No").
      </li>
      <li>
        <strong>TF.TFLink.ortho:</strong> <i>UniProt IDs</i> of ortholog transcription factors that
        are available at the TFLink gateway. Each entry consists of a shortened name of the organism
        (Hs: <i>Homo sapiens</i>,
        Mm: <i>Mus musculus</i>, Rn: <i>Rattus norvegicus</i>, Dr: <i>Danio rerio</i>, Dm:
        <i>Drosophila
          melanogaster</i>, Ce:
        <i>Caenorhabditis elegans</i>, Sc: <i>Saccharomyces cerevisiae</i>) and the UniProt ID
        separated by a colon (<i>e.g.</i> Mm:Q3UPW2).
      </li>
      <li>
        <strong>TF.nonTFLink.ortho:</strong> <i>UniProt IDs</i> of ortholog transcription factors
        that are not available at the TFLink gateway.
      </li>
      <li>
        <strong>Target.TFLink.ortho:</strong> <i>UniProt IDs</i> of ortholog target genes that are
        available at the TFLink gateway.
      </li>
      <li>
        <strong>Target.nonTFLink.ortho:</strong> <i>UniProt IDs</i> of ortholog target genes that
        are not available at the TFLink gateway.
      </li>
    </ol>
    <hr style="height: 1px;">
    * If the interaction table was downloaded at the `Download` section the file contains
    both TF and
    <i>Target IDs</i> and names. If it was downloaded from an entry page, it contains names and IDs
    of the
    TF (when downloading "Transcription factors of …" or names and IDs of the targets (when
    downloading "Targets of …")
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-mitab"></a>
    <h4>What do <i>MITAB</i> tables contain?</h4>
    <div class="justified">
      <p>
        Interaction <i>MITABs</i> contain transcription factor - target gene interactions
        in HUPO-PSI <i>MITAB 2.8</i> format. <i>MITAB 2.8</i> (as defined by the Human Proteome
        Organization
        - Proteomics Standards Initiative, HUPO-PSI) is a standardised format – including
        standardised
        vocabulary also – used to describe molecular interactions. While other databases may refer
        to
        the detection methods by multiple different names, databases that use the <i>MITAB</i>
        format
        (<i>e.g.</i>
        TFLink, <a href="https://mint.bio.uniroma2.it/" target="_blank">MINT</a> or <a
          href="https://www.ebi.ac.uk/intact/home" target="_blank">IntAct</a>) use the
        same <i>code</i> for a given technique. For example, the
        <i>electrophoretic mobility shift assay</i> technique could be identified by both its full
        name, and
        the shortened name EMSA, but in databases utilising the <i>MITAB</i> format it's always
        referred to
        by the <i>psi-mi:”MI:0413”</i> code. This makes the identification of interaction properties
        more
        efficient and helps avoid potential misunderstandings. The <i>MITAB</i> files are
        tab-delimited
        tables containing 46 columns and no header.
      </p>
      <p>
        A header for the <i>MITAB</i> tables is available <a href="/data/mitab_header.tsv"
          target="_blank">here</a>.
      </p>
      <p>
        The interaction tables and <i>MITAB</i> files can be used as input data for the <a
          href="https://cytoscape.org/" target="_blank">Cytoscape</a> software
        to perform systems and network biology studies.
      </p>
      <p>
        For more information on the <i>HUPO-PSI</i>'s molecular interaction format see: <a
          href="https://psicquic.github.io/MITAB28Format.html" target="_blank">link</a>
        Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of the
        <i>HUPO-PSI</i>
        format for molecular interactions. <i>BMC Bioinformatics</i> <strong>19</strong>(1):134. <a
          href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2118-1"
          target="_blank">
          doi: 10.1186/s12859-018-2118-1.
        </a>
      </p>
    </div>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-interaction-gmt"></a>
    <h4>What do interaction <i>GMT</i> files contain?</h4>
    <div class="justified">
      <p>
        Interaction <i>GMT</i> (Gene Matrix Transposed) is a tab delimited file format that
        describes gene
        sets – target genes of a transcription factor – in each row. The first and second column
        contains information about the transcription factors (various IDs and gene names). The first
        cell in each row is always unique. From the third to the last column the target genes of the
        transcription factor are listed. The number of target genes can vary from transcription
        factor to transcription factor, therefore the number of cells can be different in every row.
        The user can choose between <i>GMT</i> files with <i>Uniprot IDs</i>, <i>NCBI Gene IDs</i>,
        and <i>gene names</i>. The
        <i>GMT</i> files are useful for enrichment and gene overrepresentation analyses and can be
        an input
        file for the <a href="https://github.com/koralgooll/MulEA" target="_blank">MulEA</a> and <a
          href="https://www.gsea-msigdb.org/gsea/index.jsp" target="_blank">GSEA</a> software.
      </p>
    </div>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-tables-contain"></a>
    <h4>
      What do binding site tables contain?
    </h4>
    <p class="justified">
      Binding site table files are tab separated tables (<i>TSV</i>) of binding site annotations
      that contain:
    <ol>
      <li><strong>TFLinkID:</strong> unique <i>TFLink IDs</i> of the binding sites,</li>
      <li><strong>UniprotID.TF:</strong> <i>Uniprot IDs</i> of the transcription factors,</li>
      <li><strong>Name.TF:</strong> gene names of the transcription factors,</li>
      <li><strong>Organism:</strong> scientific name of the organism,</li>
      <li><strong>Assembly:</strong> version of the genome assembly,</li>
      <li><strong>Chromosome:</strong> name of the chromosome,</li>
      <li><strong>Start:</strong> start coordinates of the binding sites,</li>
      <li><strong>End:</strong> end coordinates of the binding sites,</li>
      <li><strong>Strand:</strong> coding strand ("+" indicates the forward strand, and "-" the
        reverse strand),</li>
      <li><strong>Genome.browser:</strong> a hyperlink to the particular genomic location at the
        <a href="https://genome.ucsc.edu/" target="_blank">UCSC genome browser</a>
        website,
      </li>
      <li><strong>Detection.method:</strong> names of the detection methods,</li>
      <li><strong>PubmedID:</strong> <i>Pubmed IDs</i> of the original publications (when available)
        and the publications of the databases,</li>
      <li><strong>Source.database:</strong> names of the original source databases,</li>
      <li><strong>Small-scale.evidence:</strong> indication about if the data were confirmed by
        small-scale evidence
        (with "Yes" or "No"),</li>
      <li><strong>Number.of.TFBS.overlaps:</strong> the number of overlapping binding sites of the
        same transcription
        factor, and</li>
      <li><strong>TFBS.overlaps:</strong> list of TFLink IDs of overlapping binding sites of the
        same transcription
        factor.</li>
    </ol>
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-gff3-binding-site-annotation-contain"></a>
    <h4>
      What do <i>GFF3</i> binding site annotation files contain?
    </h4>
    <p class="justified">
      Binding site annotation files contain:
    <ol>
      <li><strong>##sequence-region …:</strong> sequence regions with the name, start and end site
        of chromosomes,</li>
      <li><strong>seqid:</strong> names of the chromosomes,</li>
      <li><strong>source:</strong> starting with "TFLink_from_" and then the names of the source
        databases,</li>
      <li><strong>type:</strong> "TF_binding_site"</li>
      <li><strong>start:</strong> the start coordinates of the binding sites,</li>
      <li><strong>end: </strong>the end coordinates of the binding sites,</li>
      <li><strong>score:</strong> "."</li>
      <li><strong>strand:</strong> the coding strand ("+" indicates the forward strand, and "-" the
        reverse strand),
      </li>
      <li><strong>phase:</strong> "."</li>
      <li><strong>attributes:</strong></li>
      <ol type="a">
        <li><strong>ID:</strong> unique <i>TFLink IDs</i> of the binding sites,</li>
        <li><strong>Name:</strong> names of the transcription factors, and</li>
        <li><strong>Note:</strong> <i>Uniprot IDs</i> of the transcription factors</li>
      </ol>
    </ol>
    in <i>GFF3</i> format. For detailed description of the format, please visit this <a
      href="https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md"
      target="_blank">site</a>.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-sequence-files-contain"></a>
    <h4>
      What do binding site sequence files contain?
    </h4>
    <p class="justified">
      Binding site sequence files are <i>FASTA</i> files containing the DNA sequences of the
      transcription
      factor binding sites. The header of each sequence contains
    <ol>
      <li>unique internal <i>TFLink ID</i> of the binding site,</li>
      <li><i>Uniprot ID</i> and gene name of the transcription factor,</li>
      <li>version of the genome assembly,</li>
      <li>name of the chromosome, and</li>
      <li>start and end coordinates of the sites.</li>
    </ol>
    </p>
  </div>

  <a class="anchor" id="faq_what-source-databases"></a>
  <h3>
    What were the source databases that provided the data of <i>TFLink</i>?
  </h3>
  <u>We use the following abbreviations in the table:</u>
  <br />
  Data based on small-scale experiments: SS
  <br />
  Data based on large-scale experiments: LS
  <br />
  <i>Homo sapiens</i>: Hs
  <br />
  <i>Mus musculus</i>: Mm
  <br />
  <i>Rattus norvegicus</i>: Rn
  <br />
  <i>Danio rerio</i>: Dr
  <br />
  <i>Drosophila melanogaster</i>: Dm
  <br />
  <i>Caenorhabditis elegans</i>: Ce
  <br />
  <i>Saccharomyces cerevisiae</i>: Sc
  <br />

  <table class="faq-table uk-table uk-table-divider uk-table-small">
    <thead>
      <tr>
        <td colspan="3"></td>
        <td colspan="3">Nr. of data downloaded</td>
      </tr>
      <tr>
        <td></td>
        <td>Version</td>
        <td>Downloading date</td>
        <td>Type of data</td>
        <td>Nr. of integrated data</td>
        <td>Species</td>
      </tr>
    </thead>
    <tbody>
      
      <tr>
        
        <td rowspan=1>
          <a href="https://saezlab.github.io/dorothea/" target="_blank" >
            DoRothEA
          </a>
        </td>
        
        
        
        <td rowspan=1>2</td>
        
        
        
        <td rowspan=1>19/06/2020</td>
        
        
        <td>SS interactions</td>
        <td>3,453</td>
        
        <td rowspan=1>Hs</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=1>
          <a href="http://gtrd.biouml.org/" target="_blank" >
            GTRD
          </a>
        </td>
        
        
        
        <td rowspan=1>20.06</td>
        
        
        
        <td rowspan=1>02/07/2020</td>
        
        
        <td>LS interactions</td>
        <td>10,685,122</td>
        
        <td rowspan=1>Hs, Mm, Rn, Dr, Dm, Ce, Sc</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=2>
          <a href="http://www.lbbc.ibb.unesp.br/htri/" target="_blank" >
            HTRIdb
          </a>
        </td>
        
        
        
        <td rowspan=2>1</td>
        
        
        
        <td rowspan=2>29/04/2017</td>
        
        
        <td>SS interactions</td>
        <td>2,020</td>
        
        <td rowspan=2>Hs</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS interactions</td>
        <td>47,140</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=2>
          <a href="http://jaspar.genereg.net/" target="_blank" >
            JASPAR
          </a>
        </td>
        
        
        
        <td rowspan=2>2020</td>
        
        
        
        <td rowspan=2>22/07/2020</td>
        
        
        <td>SS binding sites</td>
        <td>3,048</td>
        
        <td rowspan=2>Hs, Mm, Rn, Dm, Ce</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS binding sites</td>
        <td>8,567,469</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=4>
          <a href="http://www.oreganno.org/" target="_blank" >
            ORegAnno
          </a>
        </td>
        
        
        
        <td rowspan=4>3.0</td>
        
        
        
        <td rowspan=4>24/05/2017</td>
        
        
        <td>SS interactions</td>
        <td>1,979</td>
        
        <td rowspan=4>Hs, Mm, Rn, Dm, Ce, Sc</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS interactions</td>
        <td>160,096</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>SS binding sites</td>
        <td>47,304</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS binding sites</td>
        <td>705,121</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=4>
          <a href="http://redfly.ccr.buffalo.edu/" target="_blank" >
            REDfly
          </a>
        </td>
        
        
        
        <td rowspan=4>6.0.2</td>
        
        
        
        <td rowspan=4>16/06/2020</td>
        
        
        <td>SS interactions</td>
        <td>683</td>
        
        <td rowspan=4>Dm</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS interactions</td>
        <td>90</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>SS binding sites</td>
        <td>2,240</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS binding sites</td>
        <td>27</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=1>
          <a href="http://remap.univ-amu.fr/" target="_blank" >
            ReMap
          </a>
        </td>
        
        
        
        <td rowspan=1>1.2</td>
        
        
        
        <td rowspan=1>16/07/2018</td>
        
        
        <td>LS interactions</td>
        <td>2,933,177</td>
        
        <td rowspan=1>Hs</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=1>
          <a href="http://rulai.cshl.edu/cgi-bin/TRED/tred.cgi?process=home" target="_blank" >
            TRED
          </a>
        </td>
        
        
        
        <td rowspan=1>-</td>
        
        
        
        <td rowspan=1>08/06/2018</td>
        
        
        <td>SS interactions</td>
        <td>8,693</td>
        
        <td rowspan=1>Hs, Mm</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=1>
          <a href="https://www.grnpedia.org/trrust/" target="_blank" >
            TRRUST
          </a>
        </td>
        
        
        
        <td rowspan=1>2</td>
        
        
        
        <td rowspan=1>30/07/2018</td>
        
        
        <td>SS interactions</td>
        <td>16,570</td>
        
        <td rowspan=1>Hs, Mm</td>
        
      </tr>
      
      <tr>
        
        <td rowspan=2>
          <a href="http://www.yeastract.com/" target="_blank" >
            Yeastract
          </a>
        </td>
        
        
        
        <td rowspan=2>2020</td>
        
        
        
        <td rowspan=2>20/07/2020</td>
        
        
        <td>SS interactions</td>
        <td>5,349</td>
        
        <td rowspan=2>Sc</td>
        
      </tr>
      
      <tr>
        
        
        
        
        
        
        <td>LS interactions</td>
        <td>188,072</td>
        
      </tr>
      
    </tbody>
  </table>

  <a class="anchor" id="faq_what-kind-of-biologiacal-questions"></a>
  <h3>What kind of biological questions can be answered by using data on TFLink?
  </h3>
  <div>
    <h4>General applications</h4>
    <p>
      TFLink is a useful resource for wet-lab researchers, since it provides easy access to high
      quality transcription factor - target gene interaction and transcription factor binding site
      data, with cross-links to several other databases. TFLink is also a long-awaited resource for
      bioinformaticians, as it contains large quantities of standardised, downloadable regulatory
      data in multiple formats. The provided interaction tables can be used as input data for the
      <a href="https://cytoscape.org/" target="_blank">Cytoscape</a> software or to the <a
        href="https://igraph.org/" target="_blank">igraph</a>
      package to perform systems and network biology studies.
      The <i>GMT</i> files are useful for gene set enrichment and overrepresentation analyses.
      Binding site
      tables allow the user to investigate the genomic location of binding sites. Users can apply
      the <i>GFF3</i> binding site annotation files in various NGS analyses, for example when
      investigating
      the mapped RNA-seq reads with <a href="https://software.broadinstitute.org/software/igv/"
        target="_blank">IGV
        genome viewer</a>. The binding site sequences can be applied in
      binding site predictions, binding site matrix calculations, as well as for investigations of
      the rate of evolution of transcription factor binding sites. Therefore, TFLink will facilitate
      benchmarking experiments in several fields of gene regulation research.
    </p>
    <h4>Use cases</h4>
    <p>
      To facilitate the application of TFLink, we provide some examples on how to use and process
      the data available at the gateway in form of <i>descriptions</i>, <i>R scripts</i> and <i>unix
        shell commands</i>:
    </p>
    <h5><a href="/data/use_cases/TFLink_Use_Case_1_Common_Taregt_Genes_of_2_TFs.html"
        target="_blank">Use case 1</a></h5>
    <p>
      Here we want to check and visualise the common target genes of two transcription factors. We
      describe how to find transcription factors which share common target genes. We cluster
      transcription factors based on their common target genes. We create a transcription factor -
      target gene interaction graph of the STAT5A and STAT5B transcription factors using the <a
        href="https://igraph.org/" target="_blank">igraph</a> R
      package. We also show how to create the same transcription factor - target gene interaction
      graph by using the <a href="https://cytoscape.org/" target="_blank">Cytoscape</a> software.
    </p>
    <h5><a href="/data/use_cases/TFLink_Use_Case_2_Between_Species_Comparison.html"
        target="_blank">Use case 2</a></h5>
    <p>
      Here we investigate the functional diversity of target genes of a nuclear hormone receptor
      transcription factor, the unc-55 in human and a nematode species. We perform <a
        href="http://geneontology.org/" target="_blank">Gene Ontology</a>
      overrepresentation analyses of the target genes in the two species in order to identify shared
      functional roles that likely represent the ancestral function of unc-55. Furthermore, this
      comparison will yield insights into the potentially divergent roles unc-55 play in these two
      distant animal groups.
    </p>
    <h5><a href="/data/use_cases/TFLink_Use_Case_3_Binding_Sites_of_EGR1.html" target="_blank">Use
        case 3</a></h5>
    <p>
      Here we investigate the binding sites of the EGR1 transcription factor. After converting the
      TFLink binding site table to <a href="https://en.wikipedia.org/wiki/BED_(file_format)"
        target="_blank">BED</a> and <a href="https://en.wikipedia.org/wiki/Binary_Alignment_Map"
        target="_blank">BAM</a> files, we calculate the "coverage" to reveal the
      strength of evidence (number of supporting experiments) for each binding site. Then we plot
      the binding sites on the human chromosomes, indicating the number of supporting evidences each
      binding site has. Finally, we investigate specific binding sites using the <a
        href="https://software.broadinstitute.org/software/igv/" target="_blank">IGV genome
        viewer</a> tool.
    </p>
  </div>

  <a class="anchor" id="faq_is-tflink-freely-available"></a>
  <h3>Is <i>TFLink</i> freely available?</h3>
  <p><i>TFLink</i> is freely available for non-commercial use.</p>
</div>

<script>
  $('#nav_faq').addClass('navbar-active');
</script>



<script src="/scrollUp.e9931.js"></script>



  </div>

  
  <footer class="uk-container uk-container-small"></footer>

  

  
  
  <script src="/main.e9931.js"></script>
  
</body>

</html>