{{ define "main" }}

{{ $statistics_faq_file := "/static/data/statistics_faq.csv" }}
{{ $sep := ";" }}
{{ $statistics_faq := getCSV $sep $statistics_faq_file }}

<div class="uk-container uk-container-small">
  <span style="text-align: center">
    <h1>FAQ</h1>
  </span>
  <hr />

  <i href="javascript:" id="return-to-top"><span uk-icon="icon: chevron-up; ratio: 2"></span></i>

  <a href="faq/#faq_who-worked" uk-scroll>Who worked on the TFLink project?</a>
  <br />
  <a href="faq/#faq_what-does-an-entry-page-contain" uk-scroll>What does an entry page contain?</a>
  <br />
  <a href="faq/#faq_what-does-a-transcription-factor-entry-page-contain" uk-scroll>What does a transcription factor
    entry page contain?</a>
  <br />
  <a href="faq/#faq_what-does-a-target-gene-entry-page-contain" uk-scroll>What does a target gene entry page
    contain?</a>
  <br />
  <a href="faq/#faq_what-does-a-transcription-factor-and-target-gene-entry-page-contain" uk-scroll>What does a
    transcription factor and target gene entry page contain?</a>
  <br />
  <a href="faq/#faq_what-are-the-downloadable-data-formats" uk-scroll>What are the downloadable data formats?</a>
  <br />
  <a href="faq/#faq_what-do-interaction-tables-contain" uk-scroll>What do interaction tables contain?</a>
  <br />
  <a href="faq/#faq_what-do-mitab" uk-scroll>What do MITAB tables contain?</a>
  <br />
  <a href="faq/#faq_what-is-mitab" uk-scroll>What is the MITAB format?</a>
  <br />
  <a href="faq/#faq_what-do-GMT-files-contain" uk-scroll>What do GMT files contain?</a>  
  <br />
  <a href="faq/#faq_what-do-binding-site-tables-contain" uk-scroll>What do binding site tables contain?</a>
  <br />
  <a href="faq/#faq_what-do-binding-site-sequence-files-contain" uk-scroll>What do binding site sequence files
    contain?</a>
  <br />
  <a href="faq/#faq_what-source-databases" uk-scroll>What source databases were used to create TFLink?</a>
  <br />
  <a href="faq/#faq_is-tflink-freely-available" uk-scroll>Is TFLink freely available?</a>

  <a class="anchor" id="faq_who-worked"></a>
  <h3>Who worked on the <i>TFLink</i> project?</h3>

  <div class="uk-card uk-card-default">
    <strong>Orsolya Liska</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:liska.orsolya@brc.hu">liska.orsolya@brc.hu</a><br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#orsolya-liska-sarkadi"
      target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
        Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
        Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/" 
        target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Bohár</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:bbazsi41@gmail.com">bbazsi41@gmail.com</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>András Hidas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:hidas.andras@ecolres.hu">hidas.andras@ecolres.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://ecolres.hu/en/Andras.HIDAS" target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://voi.ecolres.hu/en/node/14728" target="_blank">
          Institute of Aquatic Ecology, Centre for Ecological Research, Budapest,
          Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Tamás Korcsmáros</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:Tamas.Korcsmaros@earlham.ac.uk">Tamas.Korcsmaros@earlham.ac.uk</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://www.earlham.ac.uk/korcsmaros-group" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
      <li>
        <a href="https://quadram.ac.uk/" target="_blank">
          Quadram Institute, Norwich UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Papp</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:papp.balazs@brc.hu">papp.balazs@brc.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/" 
        target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Dávid Fazekas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:fazekas@netbiol.elte.hu">fazekas@netbiol.elte.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://www.earlham.ac.uk/david-fazekas" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Eszter Ari</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/" 
        target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <a class="anchor" id="faq_what-does-an-entry-page-contain"></a>
  <h3>
    What does an entry page contain?
  </h3>
  <p class="justified">
    When you select a transcription factor or a target gene from the
    Browsing table an entry page will be opened.
    On each entry page the user can find basic information about the gene or 
    protein: its gene name, <i>Uniprot ID</i> (linked to the corresponding 
    Uniprot protein page), <i>NCBI Gene ID</i> (linked to the corresponding 
    NCBI Gene database webpage), organism (the scientific name of the species), 
    its function (transcription factor, target gene or both), the number of its 
    interactions, and its orthologs (by the species name and its 
    <i>Uniprot ID</i>) – if there is any. If the ortholog is also available at 
    the TFLink database a link is provided to its TFLink entry page. 
    Each entry page contains an interactive network visualization of 
    interactions between the transcription factors and target genes.
    If there are more than 100 interactions available for a particular entry in 
    TLink, we only show the first 100 targets / transcription factors / binding 
    sites / binding site sequences on the website, but we make the full 
    information available in the form of downloadable <i>tables</i> and 
    <i>fasta</i> files.
  </p>

  <div class="margin-left">
    <a class="anchor" id="faq_what-does-a-transcription-factor-entry-page-contain"></a>
    <h4>
      What does a transcription factor entry page contain?
    </h4>
    <p class="justified">
      Besides basic information there is a <i>table</i> that contains 
      information about the target genes of the transcription factor. If there 
      are known binding sites of the transcription factor you will find a 
      <i>table</i> containing their genomic locations and – for the small-scale 
      experiments – a list containing their DNA sequences in <i>fasta</i> 
      format below the table. The <i>fasta</i> files of the bindings sites are 
      also downloadable for the small- and the large-scale experiments 
      separately. (In some cases – among the data we downloaded from the 
      <a href="http://jaspar.genereg.net/" target="_blank">JASPAR</a>
      database – there are binding sequences with missing localization, 
      for example when random sequences were investigated with the SELEX.) 
      For an example please visit: 
      <a href="https://tflink.net/protein/q9vhm6/" target="_blank">https://tflink.net/protein/q9vhm6/</a>
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-does-a-target-gene-entry-page-contain"></a>
    <h4>
      What does a target gene entry page contain?
    </h4>
    <p class="justified">
      Besides basic information there is a <i>table</i> that contains 
      information about the transcription factors of the target gene. For an 
      example please visit: 
      <a href="https://tflink.net/protein/p52429/" target="_blank">https://tflink.net/protein/p52429/</a>
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-does-a-transcription-factor-and-target-gene-entry-page-contain"></a>
    <h4>
      What does a transcription factor and target gene entry page contain?
    </h4>
    <p class="justified">
      Some proteins act like transcription factors and target genes 
      simultaneously. In some cases a transcription factor can regulate its own 
      gene too. These entry pages contain similar information as simple 
      transcription factor entry pages. 
      For an example please visit: 
      <a href="https://tflink.net/protein/p10242/" target="_blank">https://tflink.net/protein/p10242/</a>
    </p>
  </div>

  <a class="anchor" id="faq_what-are-the-downloadable-data-formats"></a>
  <h3>
    What are the downloadable data formats?
  </h3>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-interaction-tables-contain"></a>
    <h4>
      What do interaction tables contain?
    </h4>
    <p class="justified">
      Interaction table files are comma separated tables
      <i>(csv)</i> of transcription factor - target gene interactions that
      contain either interactions validated by small-scale experiments
      or large-scale experiments or these two data altogether. 
      All tables contain the following data: <i>Uniprot IDs</i>,
      gene names, <i>NCBI Gene IDs</i> of transcription factors and
      target genes, names of the detection methods,
      <i>Pubmed IDs</i>
      of the original publications, name of the organism, source
      databases, and indication about if the data were confirmed by 
      small-scale evidences.
      The interaction table files can be used as input 
      data for the 
      <a href="https://cytoscape.org/" target="_blank">Cytoscape</a>
      software to perform systems and network biology studies.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-mitab"></a>
    <h4>What do <i>MITAB</i> tables contain?</h4>
    <p class="justified">
      Interaction MITABs contain transcription factor - target gene interactions
      in HUPO-PSI MITAB 2.8 format.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-is-mitab"></a>
    <h4>What is the <i>MITAB</i> format?</h4>
    <p class="justified">
      <i>MITAB 2.8</i> is a standardized format introduced by the
      <i>Human Proteome Organization</i> (<i>HUPO</i>) which captures binary
      molecular interactions using standardized content and vocabulary. The
      <i>MITAB</i> files are tab-delimited tables containing 46 columns. A header
      for the <i>MITAB</i> tables is available
      <a href="/data/mitab_header.tsv" target="_blank">here</a>. <br />
      For more information on the <i>HUPO-PSI</i>'s molecular interaction format
      see:
      <a href="https://psicquic.github.io/MITAB28Format.html" target="_blank">link</a>
      <br />
      Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of
      the <i>HUPO-PSI</i> format for molecular interactions.
      <i>BMC Bioinformatics</i> <strong>19</strong>(1):134.
      <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2118-1" target="_blank">doi:
        10.1186/s12859-018-2118-1</a>.
      The <i>MITAB</i> table files can be used as input 
      data for the 
      <a href="https://cytoscape.org/" target="_blank">Cytoscape</a>
      software to perform systems and network biology studies.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-GMT-files-contain"></a>
    <h4>
      What do <i>GMT</i> files contain?
    </h4>
    <p class="justified">
      Interaction <i>GMT</i> (Gene Matrix Transposed) is a tab 
      delimited file format that describes gene sets – target genes of a 
      transcription factor – in each row. The first and second column 
      contains information about the transcription factors (various IDs and 
      gene names). The first cell in each row is always unique. From the 
      third to the last column the target genes of the transcription factor 
      are listed. The number of target genes can vary from transcription 
      factor to transcription factor, therefore the number of cells can be 
      different in every row. The user can choose between <i>GMT</i> files 
      with <i>Uniprot IDs</i>, <i>NCBI Gene IDs</i>, and <i>gene names</i>. 
      The <i>GMT</i> files are useful for enrichment and gene 
      overrepresentation analyses and can be an input file for the 
      <a href="https://github.com/koralgooll/MulEA" target="_blank">MulEA</a> 
      and 
      <a href="https://www.gsea-msigdb.org/gsea/index.jsp" target="_blank">GSEA</a> 
      software.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-tables-contain"></a>
    <h4>
      What do binding site tables contain?
    </h4>
    <p class="justified">
      Binding site table files are comma separated tables 
      (<i>csv</i>) of binding site annotations that contain unique 
      <i>TFLink IDs</i> of binding sites, <i>Uniprot IDs</i> and gene names 
      of the transcription factors, the names of the detection method, 
      indication if it is a small- or a large-scale method, <i>Pubmed IDs</i> 
      of the original publications, the organism, the source database, the 
      version of the genome assembly, the name of the chromosome, the start 
      and end coordinates of the sites, and the coding strand.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-sequence-files-contain"></a>
    <h4>
      What do binding site sequence files contain?
    </h4>
    <p class="justified">
      Binding site sequence files are <i>fasta</i> files 
      containing the DNA sequences of the transcription factor binding sites. 
      The header of each sequence contains the unique internal 
      <i>TFLink ID</i> of the binding site, the <i>Uniprot ID</i> and gene 
      name of the transcription factor, the version of the genome assembly, 
      the name of the chromosome, and the start and end coordinates of the 
      sites.
    </p>
  </div>

  <a class="anchor" id="faq_what-source-databases"></a>
  <h3>
    What source databases were used to create TFLink?
  </h3>
  <u>We use the following abbreviations in the table:</u>
  <br />
  Data based on small-scale experiments: SS
  <br />
  Data based on large-scale experiments: LS
  <br />
  <i>Homo sapiens</i>: Hs
  <br />
  <i>Mus musculus</i>: Mm
  <br />
  <i>Rattus norvegicus</i>: Rn
  <br />
  <i>Danio rerio</i>: Dr
  <br />
  <i>Drosophila melanogaster</i>: Dm
  <br />
  <i>Caenorhabditis elegans</i>: Ce
  <br />
  <i>Saccharomyces cerevisiae</i>: Sc
  <br />

  <table class="faq-table uk-table uk-table-divider uk-table-small">
    <thead>
      <tr>
        <td colspan="3"></td>
        <td colspan="3">Nr. of data downloaded</td>
      </tr>
      <tr>
        <td></td>
        <td>Version</td>
        <td>Downloading date</td>
        <td>Type of data</td>
        <td>Nr. of integrated data</td>
        <td>Species</td>
      </tr>
    </thead>
    <tbody>
      {{ range $row := (after 1 $statistics_faq) }}
        <tr>
          {{ with index $row 0 }}
            <td rowspan={{ . }}>
              <a {{ printf `href=%q target="_blank"` (index $row 1) | safeHTMLAttr }}>
                {{ index $row 2 }}
              </a>
            </td>
          {{ end }}
          {{ range $cell := first 2 (after 3 .) }}
            {{ with index $row 0 }}
              <td rowspan={{ . }}>{{ $cell }}</td>
            {{ end }}
          {{ end }}
          <td>{{ index $row 5 }}</td>
          <td>{{ lang.NumFmt 0 (int (index $row 6)) "- . ," }}</td>
          {{ with index $row 0 }}
              <td rowspan={{ . }}>{{ index $row 7 }}</td>
          {{ end }}
        </tr>
      {{ end }} 
    </tbody>
  </table>

  <a class="anchor" id="faq_is-tflink-freely-available"></a>
  <h3>Is <i>TFLink</i> freely available?</h3>
  <p><i>TFLink</i> is freely available for non-commercial use.</p>
</div>

<script>
  $('#nav_faq').addClass('navbar-active');
</script>

{{ $script := .Site.Data.webpack.scrollUp }}
{{ with $script.js }}
  <script src="{{ relURL . }}"></script>
{{ end }}

{{ end }}
