{{ define "main" }}

{{ $statistics_faq_file := "/static/data/statistics_faq.csv" }}
{{ $sep := ";" }}
{{ $statistics_faq := getCSV $sep $statistics_faq_file }}

<div class="uk-container uk-container-small">
  <span style="text-align: center">
    <h1>FAQ</h1>
  </span>
  <hr />

  <i href="javascript:" id="return-to-top"><span uk-icon="icon: chevron-up; ratio: 2"></span></i>

  <a href="faq/#faq_who-worked" uk-scroll>Who worked on the <i>TFLink</i> project?</a>
  <br />
  <a href="faq/#faq_what-does-an-entry-page-contain" uk-scroll>What does an entry page contain?</a>
  <div class="margin-left">
    <a href="faq/#faq_what-does-a-transcription-factor-entry-page-contain" uk-scroll>What does a
      transcription factor
      entry page contain?</a>
    <br />
    <a href="faq/#faq_what-does-a-target-gene-entry-page-contain" uk-scroll>What does a target gene
      entry page
      contain?</a>
    <br />
    <a href="faq/#faq_what-does-a-transcription-factor-and-target-gene-entry-page-contain"
      uk-scroll>What does a
      transcription factor and target gene entry page contain?</a>
  </div>
  <a href="faq/#faq_what-are-the-downloadable-data-formats" uk-scroll>What are the downloadable data
    formats?</a>
  <div class="margin-left">
    <a href="faq/#faq_what-do-interaction-tables-contain" uk-scroll>What do interaction tables
      contain?</a>
    <br />
    <a href="faq/#faq_what-do-mitab" uk-scroll>What do <i>MITAB</i> tables contain?</a>
    <br />
    <a href="faq/#faq_what-is-mitab" uk-scroll>What is the <i>MITAB</i> format?</a>
    <br />
    <a href="faq/#faq_what-do-binding-site-tables-contain" uk-scroll>What do binding site tables
      contain?</a>
    <br />
    <a href="faq/#faq_what-do-binding-site-annotation-files-contain" uk-scroll>What do binding site
      annotation files
      contain?</a>
    <br />
    <a href="faq/#faq_what-do-binding-site-sequence-files-contain" uk-scroll>What do binding site
      sequence files
      contain?</a>
  </div>
  <a href="faq/#faq_what-source-databases" uk-scroll>What source databases provided the data of
    <i>TFLink</i>?</a>
  <br />
  <a href="faq/#faq_what-kind-of-biological-questions" uk-scroll>What kind of biological questions
    can be answered by
    using data on
    <i>TFLink</i>?</a>
  <br />
  <a href="faq/#faq_is-tflink-freely-available" uk-scroll>Is <i>TFLink</i> freely available?</a>

  <a class="anchor" id="faq_who-worked"></a>
  <h3>Who worked on the <i>TFLink</i> project?</h3>

  <div class="uk-card uk-card-default">
    <strong>Orsolya Liska</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:liska.orsolya@brc.hu">liska.orsolya@brc.hu</a><br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#orsolya-liska-sarkadi"
      target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Bohár</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:bbazsi41@gmail.com">bbazsi41@gmail.com</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>András Hidas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:hidas.andras@ecolres.hu">hidas.andras@ecolres.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://ecolres.hu/en/Andras.HIDAS" target="_blank">website</a><br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://voi.ecolres.hu/en/node/14728" target="_blank">
          Institute of Aquatic Ecology, Centre for Ecological Research, Budapest,
          Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Tamás Korcsmáros</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:Tamas.Korcsmaros@earlham.ac.uk">Tamas.Korcsmaros@earlham.ac.uk</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://www.earlham.ac.uk/korcsmaros-group" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
      <li>
        <a href="https://quadram.ac.uk/" target="_blank">
          Quadram Institute, Norwich UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Balázs Papp</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:papp.balazs@brc.hu">papp.balazs@brc.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html"
      target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Dávid Fazekas</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:fazekas@netbiol.elte.hu">fazekas@netbiol.elte.hu</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="https://www.earlham.ac.uk/david-fazekas" target="_blank">website</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
          Earlham Institute, Norwich, UK
        </a>
      </li>
    </ul>
  </div>

  <div class="uk-card uk-card-default">
    <strong>Eszter Ari</strong><br />
    <span uk-icon="icon: mail"></span>
    <a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a>
    <br />
    <span uk-icon="icon: laptop"></span>
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari"
      target="_blank">website
      1</a>,
    <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148" target="_blank">website 2</a>
    <br />
    <span uk-icon="icon: bookmark"></span>
    Affiliations:
    <ul>
      <li>
        <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
          Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
          Research Centre, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://www.hcemm.eu/research-groups/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
          target="_blank">
          Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
        </a>
      </li>
      <li>
        <a href="https://genet.elte.hu/" target="_blank">
          Department of Genetics, Eötvös Loránd University, Budapest, Hungary
        </a>
      </li>
    </ul>
  </div>

  <a class="anchor" id="faq_what-does-an-entry-page-contain"></a>
  <h3>
    What does an entry page contain?
  </h3>
  <p class="justified">
    When you select a transcription factor or a target gene protein from the Browsing table a
    protein entry page will be opened.
  </p>

  <div class="margin-left">
    <a class="anchor" id="faq_what-does-a-transcription-factor-entry-page-contain"></a>
    <h4>
      What does a transcription factor entry page contain?
    </h4>
    <p class="justified">
      Entry pages of transcription factors contain their basic information (name, organism, number
      of interactions) as well as links to the UniProt and NCBI Gene pages of the protein and its
      coding gene, respectively. Below, you will find a table that contains information about the
      target genes of the transcription factor. Interactive visualization of the transcription
      factor - target gene interactions is also available.
      <br />
      If there are known binding sites of the transcription factor you will find a table containing
      their genomic locations and a list containing their DNA sequences in fasta format below the
      table.
      <br />
      Only the first 100 interactions and binding sites are shown on the entry pages. If the
      transcription factor has over 100 known interactions or binding sites, a download link will be
      provided for the full data above the table.

    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-does-a-target-gene-entry-page-contain"></a>
    <h4>
      What does a target gene entry page contain?
    </h4>
    <p class="justified">
      Entry pages of target genes contain their basic information (name, organism, number of
      interactions) as well as links to the UniProt and NCBI Gene pages of the protein and its
      coding gene, respectively. Below, you will find a table that contains information about the
      transcription factors of the target gene. Interactive visualization of the transcription
      factor - target gene interactions is also available.
      <br />
      Only the first 100 interactions are shown on the entry pages. If the target gene has
      over 100 known interactions, a download link will be provided for the full data above the
      table.

    </p>
  </div>

  <div class="margin-left">
    <a class="anchor"
      id="faq_what-does-a-transcription-factor-and-target-gene-entry-page-contain"></a>
    <h4>
      What does a transcription factor and target gene entry page contain?
    </h4>
    <p class="justified">
      Some proteins act as transcription factors and target genes simultaneously. In some cases
      a transcription factor can also regulate its own gene.
      <br />
      Besides basic information and links to the UniProt and NCBI Gene databases, the entry page
      also contains separate tables of the target genes of the protein and transcription factor of
      the coding gene. Interactive visualization of the protein's target genes and its transcription
      factors is available separately.
      <br />
      If there are known binding sites of the transcription factor you will find a table containing
      their genomic locations and a list containing their DNA sequences in fasta format below the
      table.
      <br />
      Only the first 100 interactions and binding sites are shown on the entry pages. If the
      transcription factor has over 100 known interactions or binding sites, a download link will be
      provided for the full data above the table.

    </p>
  </div>

  <a class="anchor" id="faq_what-are-the-downloadable-data-formats"></a>
  <h3>
    What are the downloadable data formats?
  </h3>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-interaction-tables-contain"></a>
    <h4>
      What do interaction tables contain?
    </h4>
    <p class="justified">
      Interaction table files are tab separated tables (<i>TSV</i>) of transcription factor - target
      gene
      interactions that contain either interactions validated by small-scale experiments or
      large-scale experiments or these two data altogether. Interaction tables contain the following
      data:
    <ol>
      <li><strong>UniprotID.TF</strong> and/or<sup>1</sup> <strong>UniprotID.Target:</strong>
        <i>Uniprot IDs</i> of
        transcription factors and/or target genes
      </li>
      <li>
        <strong>NCBI.GeneID.TF</strong> and/or <strong>NCBI.GeneID.Target:</strong> <i>NCBI Gene
          IDs</i> of
        transcription factors and/or target genes,
      </li>
      <li><strong>Name.TF</strong> and/or <strong>Name.Target:</strong> gene names of transcription
        factors and/or target genes,</li>
      <li><strong>Detection.method:</strong> names of the detection methods,</li>
      <li><strong>PubmedID:</strong> <i>Pubmed IDs</i> of the original publications,</li>
      <li><strong>Organism:</strong> scientific name of the organism,</li>
      <li><strong>Source.database:</strong> names of the original source databases, and</li>
      <li>
        <strong>Small-scale.evidence:</strong> indication about if the data were confirmed by
        small-scale evidence (with "Yes" or "No").
      </li>
      <li>
        <strong>TF.TFLink.ortho:</strong> UniProt IDs of the orthologs of transcription factors that
        are also present
        in TFLink. Each entry consists of a shortened name of the organism (Hs: <i>Homo sapiens</i>,
        Mm: <i>Mus musculus</i>, Rn: <i>Rattus norvegicus</i>, Dr: <i>Danio rerio</i>, Dm:
        <i>Drosophila
          melanogaster</i>, Ce:
        <i>Caenorhabditis elegans</i>, Sc: <i>Saccharomyces cerevisiae</i>) and the UniProt ID
        separated by a colon (e.g. Mm:Q3UPW2).
      </li>
      <li>
        <strong>TF.nonTFLink.ortho:</strong> UniProt IDs of the orthologs of transcription factors
        that are not present in TFLink.
      </li>
      <li>
        <strong>Target.TFLink.ortho:</strong> UniProt IDs of the orthologs of target genes that are
        also present in TFLink.
      </li>
      <li>
        <strong>Target.nonTFLink.ortho:</strong> UniProt IDs of the orthologs of target genes that
        are not present in TFLink.
      </li>
    </ol>
    <hr style="height: 1px;">
    <sup>1</sup> If the interaction table was downloaded at the `Download` section the file contains
    both TF and
    Target IDs and names. If it was downloaded from an entry page, it contains names and IDs of the
    TF (when downloading "Transcription factors of …" or names and IDs of the targets (when
    downloading "Targets of …")
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-mitab"></a>
    <h4>What do <i>MITAB</i> tables contain?</h4>
    <p class="justified">
      Interaction MITABs contain transcription factor - target gene interactions
      in HUPO-PSI MITAB 2.8 format.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-is-mitab"></a>
    <h4>What is the <i>MITAB</i> format?</h4>
    <p class="justified">
      <i>MITAB 2.8</i> is a standardized format introduced by the
      <i>Human Proteome Organization</i> (<i>HUPO</i>) which captures binary
      molecular interactions using standardized content and vocabulary. The
      <i>MITAB</i> files are tab-delimited tables containing 46 columns. A header
      for the <i>MITAB</i> tables is available
      <a href="/data/mitab_header.tsv" target="_blank">here</a>. <br />
      For more information on the <i>HUPO-PSI</i>'s molecular interaction format
      see:
      <a href="https://psicquic.github.io/MITAB28Format.html" target="_blank">link</a>
      <br />
      Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of
      the <i>HUPO-PSI</i> format for molecular interactions.
      <i>BMC Bioinformatics</i> <strong>19</strong>(1):134.
      <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2118-1"
        target="_blank">doi:
        10.1186/s12859-018-2118-1</a>.
      The <i>MITAB</i> table files can be used as input
      data for the
      <a href="https://cytoscape.org/" target="_blank">Cytoscape</a>
      software to perform systems and network biology studies.
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-tables-contain"></a>
    <h4>
      What do binding site tables contain?
    </h4>
    <p class="justified">
      Binding site table files are tab separated tables (TSV) of binding site annotations that
      contain:
    <ol>
      <li><strong>TFLinkID:</strong> unique <i>TFLink IDs</i> of the binding sites,</li>
      <li><strong>UniprotID.TF:</strong> <i>Uniprot IDs</i> of the transcription factors,</li>
      <li><strong>Name.TF:</strong> gene names of the transcription factors,</li>
      <li><strong>Organism:</strong> scientific name of the organism,</li>
      <li><strong>Assembly:</strong> version of the genome assembly,</li>
      <li><strong>Chromosome:</strong> name of the chromosome,</li>
      <li><strong>Start:</strong> start coordinates of the binding sites,</li>
      <li><strong>End:</strong> end coordinates of the binding sites,</li>
      <li><strong>Strand:</strong> coding strand ("+" indicates the forward strand, and "-" the
        reverse strand),</li>
      <li><strong>Genome.browser:</strong> a hyperlink to the particular genomic location at the
        UCSC genome browser
        website (<a href="https://genome.ucsc.edu/" target="_blank">https://genome.ucsc.edu/</a>),
      </li>
      <li><strong>Detection.method:</strong> names of the detection methods,</li>
      <li><strong>PubmedID:</strong> <i>Pubmed IDs</i> of the original publications,</li>
      <li><strong>Source.database:</strong> names of the original source databases,</li>
      <li><strong>Small-scale.evidence:</strong> indication about if the data were confirmed by
        small-scale evidence
        (with "Yes" or "No"),</li>
      <li><strong>Number.of.TFBS.overlaps:</strong> the number of overlapping binding sites of the
        same transcription
        factor, and</li>
      <li><strong>TFBS.overlaps:</strong> list of TFLink IDs of overlapping binding sites of the
        same transcription
        factor.</li>
    </ol>
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-annotation-files-contain"></a>
    <h4>
      What do binding site annotation files contain?
    </h4>
    <p class="justified">
      Binding site annotation files contain:
    <ol>
      <li><strong>##sequence-region …:</strong> sequence regions with the name, start and end site
        of chromosomes,</li>
      <li><strong>seqid:</strong> names of the chromosomes,</li>
      <li><strong>source:</strong> starting with "TFLink_from_" and then the names of the source
        databases,</li>
      <li><strong>type:</strong> "TF_binding_site"</li>
      <li><strong>start:</strong> the start coordinates of the binding sites,</li>
      <li><strong>end: </strong>the end coordinates of the binding sites,</li>
      <li><strong>score:</strong> "."</li>
      <li><strong>strand:</strong> the coding strand ("+" indicates the forward strand, and "-" the
        reverse strand),
      </li>
      <li><strong>phase:</strong> "."</li>
      <li><strong>attributes:</strong></li>
      <ol type="a">
        <li><strong>ID:</strong> unique TFLink IDs of the binding sites,</li>
        <li><strong>Name:</strong> names of the transcription factors, and</li>
        <li><strong>Note:</strong> Uniprot IDs of the transcription factors</li>
      </ol>
    </ol>
    in GFF3 format. For detailed description of the format, please visit:
    <a href="https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md"
      target="_blank">https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md</a>
    </p>
  </div>

  <div class="margin-left">
    <a class="anchor" id="faq_what-do-binding-site-sequence-files-contain"></a>
    <h4>
      What do binding site sequence files contain?
    </h4>
    <p class="justified">
      Binding site sequence files are <i>FASTA</i> files containing the DNA sequences of the
      transcription
      factor binding sites. The header of each sequence contains
    <ol>
      <li>unique internal <i>TFLink ID</i> of the binding site,</li>
      <li><i>Uniprot ID</i> and gene name of the transcription factor,</li>
      <li>version of the genome assembly,</li>
      <li>name of the chromosome, and</li>
      <li>start and end coordinates of the sites.</li>
    </ol>
    </p>
  </div>

  <a class="anchor" id="faq_what-source-databases"></a>
  <h3>
    What source databases were used to create TFLink?
  </h3>
  <u>We use the following abbreviations in the table:</u>
  <br />
  Data based on small-scale experiments: SS
  <br />
  Data based on large-scale experiments: LS
  <br />
  <i>Homo sapiens</i>: Hs
  <br />
  <i>Mus musculus</i>: Mm
  <br />
  <i>Rattus norvegicus</i>: Rn
  <br />
  <i>Danio rerio</i>: Dr
  <br />
  <i>Drosophila melanogaster</i>: Dm
  <br />
  <i>Caenorhabditis elegans</i>: Ce
  <br />
  <i>Saccharomyces cerevisiae</i>: Sc
  <br />

  <table class="faq-table uk-table uk-table-divider uk-table-small">
    <thead>
      <tr>
        <td colspan="3"></td>
        <td colspan="3">Nr. of data downloaded</td>
      </tr>
      <tr>
        <td></td>
        <td>Version</td>
        <td>Downloading date</td>
        <td>Type of data</td>
        <td>Nr. of integrated data</td>
        <td>Species</td>
      </tr>
    </thead>
    <tbody>
      {{ range $row := (after 1 $statistics_faq) }}
      <tr>
        {{ with index $row 0 }}
        <td rowspan={{ . }}>
          <a {{ printf `href=%q target="_blank" ` (index $row 1) | safeHTMLAttr }}>
            {{ index $row 2 }}
          </a>
        </td>
        {{ end }}
        {{ range $cell := first 2 (after 3 .) }}
        {{ with index $row 0 }}
        <td rowspan={{ . }}>{{ $cell }}</td>
        {{ end }}
        {{ end }}
        <td>{{ index $row 5 }}</td>
        <td>{{ lang.NumFmt 0 (int (index $row 6)) "- . ," }}</td>
        {{ with index $row 0 }}
        <td rowspan={{ . }}>{{ index $row 7 }}</td>
        {{ end }}
      </tr>
      {{ end }}
    </tbody>
  </table>

  <a class="anchor" id="faq_what-kind-of-biological-questions"></a>
  <h3>What kind of biological questions can be answered by using data on
    <i>TFLink</i>?
  </h3>
  <div>
    <h4>General applications</h4>
    <p>
      TFLink is a useful resource for wet-lab researchers, since it provides easy access to high
      quality transcription factor - target gene interaction and transcription factor binding site
      data, with cross-links to several other databases. TFLink is also a long-awaited resource for
      bioinformaticians, as it contains large quantities of standardized, downloadable regulatory
      data
      in multiple formats. The provided interaction tables can be used as input data for the <a
        href="https://cytoscape.org/" target="_blank">Cytoscape</a>
      software or to the <a href="https://igraph.org/" target="_blank">igraph</a> package to perform
      systems and network biology studies. The <i>GMT</i>
      files
      are useful for gene set enrichment and overrepresentation analyses. Binding site tables allow
      the user to investigate the genomic location of binding sites. Users can apply the <i>GFF3</i>
      binding
      site annotation files in various NGS analyses, for example when investigating the mapped
      RNA-seq
      reads with <a href="https://software.broadinstitute.org/software/igv/" target="_blank">IGV
        genome viewer</a>. The binding site sequences can be applied in binding site
      predictions, binding site matrix calculations, as well as for investigations of the rate of
      evolution of transcription factor binding sites. Therefore, TFLink will facilitate
      benchmarking
      experiments in several fields of gene regulation research.
    </p>
    <h4>Use cases</h4>
    <p>
      To facilitate the application of TFLink, we provide some examples on how to use and process
      the
      data available at the gateway in form of <i>R scripts</i> and <i>unix shell commands</i>:
    </p>
    <h5>Use case 1</h5>
    <p>
      Here we want to check and visualize the common target genes of two transcription factors. We
      describe how to find transcription factors which share common target genes. We cluster
      transcription factors based on their common target genes. We create a transcription factor -
      target gene interaction graph of the XXX and XXX transcription factors using the <a
        href="https://igraph.org/" target="_blank">igraph</a> R
      package. We also show how to create the same transcription factor - target gene interaction
      graph by using the <a href="https://cytoscape.org/" target="_blank">Cytoscape</a> software.
    </p>
    <h5>Use case 2</h5>
    <p>
      Here we investigate the diversity of the target genes of two specific transcription factors in
      different species. This experiment includes the comparison of the target genes of a fairly
      conservative transcription factor, NFE2L2 [40], and the comparison of the target genes of a
      rapidly evolving protein, STAT5A [41]. Furthermore, we employ <a
        href="http://geneontology.org/" target="_blank">Gene Ontology</a> enrichment
      analyses to explore the functions of these common and diverging target gene sets.
    </p>
    <h5><a
        href="https://eltehu-my.sharepoint.com/:u:/g/personal/arieszter_ttk_elte_hu/EWlu6faE44ZAtJHjfviLpv8BKSoc6603RV0y0K41WGUkrg?e=jrnjc3"
        target="_blank">Use
        case 3</a></h5>
    <p>
      Here we investigate the binding sites of the EGR1 transcription factor. After converting the
      TFLink binding site table to BED and BAM files, we calculate the "coverage" to reveal the
      strength of evidence (number of supporting experiments) for each binding site. Then we plot
      the
      binding sites on the human chromosomes, indicating the number of supporting evidences each
      binding site has. Finally, we investigate specific binding sites using the <a
        href="https://software.broadinstitute.org/software/igv/" target="_blank">IGV genome
        viewer</a> tool.
    </p>
  </div>

  <a class="anchor" id="faq_is-tflink-freely-available"></a>
  <h3>Is <i>TFLink</i> freely available?</h3>
  <p><i>TFLink</i> is freely available for non-commercial use.</p>
</div>

<script>
  $('#nav_faq').addClass('navbar-active');
</script>

{{ $script := .Site.Data.webpack.scrollUp }}
{{ with $script.js }}
<script src="{{ relURL . }}"></script>
{{ end }}

{{ end }}