---
title: "FAQ"
---

## Who was working on the TFLink project?
Orsolya Liska    
liska.orsolya@brc.mta.hu    
webpage  
Affiliations:   
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary  
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary  
Department of Genetics, Eötvös Loránd University, Budapest, Hungary  
Balázs Bohár  
bbazsi41@gmail.com  
Affiliations:  
Department of Genetics, Eötvös Loránd University, Budapest, Hungary  
Earlham Institute, Norwich, UK  
András Hidas  
hidasandris@gmail.com  
Affiliation:  
MTA Centre for Ecological Research, Danube Research Institute, Budapest, Hungary  
Tamás Korcsmáros  
Tamas.Korcsmaros@earlham.ac.uk  
webpage  
Affiliations:  
Earlham Institute, Norwich, UK  
Quadram Institute, Norwich UK  
Balázs Papp  
papp.barazs@brc.mta.hu  
webpage  
Affiliations:   
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary  
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary  
Dávid Fazekas  
fazekas@netbiol.elte.hu  
webpage  
Affiliations:  
Department of Genetics, Eötvös Loránd University, Budapest, Hungary  
Earlham Institute, Norwich, UK  
Eszter Ari  
arieszter@gmail.com  
webpage1  
webpage2  
Affiliations:   
Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary  
Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary  
Department of Genetics, Eötvös Loránd University, Budapest, Hungary  
What are the downloadable data formats?  
What do interaction tables contain?  
Interaction tables are comma separated tables (CSV) of transcription factor - target gene interactions that contains either experimentally validated interaction (experiment interaction table) or predicted interactions (predicted experiment interaction table), along with the following data: Uniprot IDs, NCBI Gene IDs and gene names of the transcription factors and the target genes; the names of the detection methods; the Pubmed IDs of the original publications; the name of the organism and the source database.  
What do MITAB tables contain?  
Interaction MITABs contain transcription factor - target gene interactions in HUPO-PSI MITAB 2.8 format.  
What is MITAB format?  
MITAB 2.8 is a standardized format introduced by the Human Proteome Organization (HUPO) which captures binary molecular interactions using standardized content and vocabulary. The MITAB files are tab-delimited tables containing 46 columns. A header for the MITAB tables is available here.   
For more information on the HUPO-PSI's molecular interaction format see: link  
Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of the HUPO-PSI format for molecular interactions. BMC Bioinformatics 19(1):134. doi: 10.1186/s12859-018-2118-1.  
What do binding site tables contain?  
Binding site tables are comma separated tables (CSV) of binding site annotations that contain the unique internal TFLink IDs of binding sites; Uniprot IDs and protein names of the transcription factors; the names of the detection method; Pubmed IDs of the original publications; the organism and the source database; the version of the genome assembly; the name of the chromosome; the start and end coordinates of site; and the strand.  
What do binding site sequence files contain?  
Binding site sequences are FASTA files containing the DNA sequences of the transcription factor binding sites. The header of each sequence contains the unique internal TFLink ID of the binding site; the Uniprot ID and protein name of the transcription factor, the version of the genome assembly; and the the start and end coordinates of site.  
What source databases provide the data of TFLink?  
We use the following abbreviations in the table:  
Data based on low-throughput experiments: LT  
Data based on high-throughput experiments: HT  
Homo sapiens: Hs  
Mus musculus: Mm  
Rattus norvegicus: Rn  
Drosophila melanogaster: Dm  
Caenorhabditis elegans: Ce  
Saccharomyces cerevisiae: Sc  
  
  
  
  
Nr. of data downloaded  
  
  
Version  
Downloading date  
Type of data  
Nr. of integrated data  
species  
  
GTRD  
  
  
  
  
interactions  
  
  
  
  
binding sites  
  
  
HTRIdb   
no longer available  
29/04/2017  
interactions  
  
  
Hs  
JASPAR  
2016  
16/08/2017  
binding sites  
  
  
Hs, Mm, Rn, Dm, Ce  
ORegAnno  
2016 (the website is out of order)  
24/05/2017  
interactions  
  
  
Hs, Mm, Rn, Dm, Ce, Sc  
  
  
binding sites  
  
  
REDfly  
5.5  
03/09/2018  
interactions  
  
  
Dm  
binding sites  
  
  
ReMap  
  
  
  
  
interactions  
  
  
  
  
binding sites  
  
  
TRED  
obtained from RegNetwork 2017  
06/08/2018  
interactions  
  
  
Hs, Mm  
TRRUST  
2  
30/07/2018  
interactions  
  
  
Hs, Mm  
Yeastract  
-  
29/06/2017  
interactions  
  
  
Sc  
  
What kind of experimental methods does TFLink contain?  
