---
title: 'FAQ'
---

<span style="text-align: center">
  <h1>FAQ</h1>
</span>
<hr />

<a href="javascript:" id="return-to-top"><span uk-icon="icon: chevron-up; ratio: 2"></span></a>

<a href="faq/#faq_who-worked" uk-scroll>Who worked on the TFLink project?</a>
<br />
<a href="faq/#faq_what-does-a-protein-page-contain" uk-scroll>What does a protein page contain?</a>
<br />
<a href="faq/#faq_what-does-a-transcription-factor-protein-page-contain" uk-scroll>What does a transcription factor
  protein page contain?</a>
<br />
<a href="faq/#faq_what-does-a-target-gene-protein-page-contain" uk-scroll>What does a target gene protein page
  contain?</a>
<br />
<a href="faq/#faq_what-does-a-transcription-factor-and-target-gene-protein-page-contain" uk-scroll>What does a
  transcription factor and target gene protein page contain?</a>
<br />
<a href="faq/#faq_what-are-the-downloadable-data-formats" uk-scroll>What are the downloadable data formats?</a>
<br />
<a href="faq/#faq_what-do-interaction-tables-contain" uk-scroll>What do interaction tables contain?</a>
<br />
<a href="faq/#faq_what-do-mitab" uk-scroll>What do MITAB tables contain?</a>
<br />
<a href="faq/#faq_what-is-mitab" uk-scroll>What is the MITAB format?</a> <br />
<a href="faq/#faq_what-do-binding-site-tables-contain" uk-scroll>What do binding site tables contain?</a>
<br />
<a href="faq/#faq_what-do-binding-site-sequence-files-contain" uk-scroll>What do binding site sequence files
  contain?</a>
<br />
<a href="faq/#faq_what-source-databases" uk-scroll>What source databases provided the data of TFLink?</a>
<br />
<a href="faq/#faq_where-can-I-find-the-the-interaction" uk-scroll>Where can I find the the interaction and binding site
  data?</a>
<br />
<a href="faq/#faq_where-can-i-find-the-interactions-small-scale" uk-scroll>Where can I find the interactions and binding
  sites based on small-scale
  experiments?</a>
<br />
<a href="faq/#faq_where-can-i-find-the-interactions-large-scale" uk-scroll>Where can I find the interactions and binding
  sites based on large-scale
  experiments?</a>
<br />
<a href="faq/#faq_where-can-i-find-the-data-of-danio-rerio" uk-scroll>Where can I find the data of Danio rerio?</a>
<br />
<a href="faq/#faq_is-tflink-freely-available" uk-scroll>Is TFLink freely available?</a>

<a class="anchor" id="faq_who-worked"></a>
<h3>Who worked on the <i>TFLink</i> project?</h3>

<div class="uk-card uk-card-default">
  <strong>Orsolya Liska</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:liska.orsolya@brc.mta.hu">liska.orsolya@brc.mta.hu</a><br />
  <span uk-icon="icon: laptop"></span>
  <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#orsolya-liska-sarkadi">webpage</a><br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>
      Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
      Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary
    </li>
    <li>
      Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
    </li>
    <li>
      Department of Genetics, Eötvös Loránd University, Budapest, Hungary
    </li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>Balázs Bohár</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:bbazsi41@gmail.com">bbazsi41@gmail.com</a> <br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>
      Department of Genetics, Eötvös Loránd University, Budapest, Hungary
    </li>
    <li>Earlham Institute, Norwich, UK</li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>András Hidas</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:hidas.andras@okologia.mta.hu">hidas.andras@okologia.mta.hu</a> <br />
  <span uk-icon="icon: laptop"></span>
  <a href="https://www.okologia.mta.hu/en/Andras.HIDAS">webpage</a><br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>
      MTA Centre for Ecological Research, Danube Research Institute, Budapest,
      Hungary
    </li>
    <li>
      Doctoral School of Environmental Sciences, Eötvös Loránd University,
      Budapest, Hungary
    </li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>Tamás Korcsmáros</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:Tamas.Korcsmaros@earlham.ac.uk">Tamas.Korcsmaros@earlham.ac.uk</a>
  <br />
  <span uk-icon="icon: laptop"></span>
  <a href="http://www.earlham.ac.uk/korcsmaros-group">webpage</a> <br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>Earlham Institute, Norwich, UK</li>
    <li>Quadram Institute, Norwich UK</li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>Balázs Papp</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:papp.balazs@brc.mta.hu">papp.balazs@brc.mta.hu</a> <br />
  <span uk-icon="icon: laptop"></span>
  <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html">webpage</a>
  <span uk-icon="icon: bookmark"></span>
  <br />
  Affiliations:
  <ul>
    <li>
      Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
      Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary
    </li>
    <li>
      Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
    </li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>Dávid Fazekas</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:fazekas@netbiol.elte.hu">fazekas@netbiol.elte.hu</a>
  <br />
  <span uk-icon="icon: laptop"></span>
  <a href="https://www.earlham.ac.uk/david-fazekas">webpage</a> <br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>
      Department of Genetics, Eötvös Loránd University, Budapest, Hungary
    </li>
    <li>Earlham Institute, Norwich, UK</li>
  </ul>
</div>

<div class="uk-card uk-card-default">
  <strong>Eszter Ari</strong><br />
  <span uk-icon="icon: mail"></span>
  <a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a> <br />
  <span uk-icon="icon: laptop"></span>
  <a href="https://genetics.elte.hu/index.php/hu/eszter-ari">webpage1</a>
  <br />
  <span uk-icon="icon: laptop"></span>
  <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari">webpage2</a>
  <br />
  <span uk-icon="icon: bookmark"></span>
  Affiliations:
  <ul>
    <li>
      Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological
      Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary
    </li>
    <li>
      Hungarian Centre of Excellence for Molecular Medicine, Szeged, Hungary
    </li>
    <li>
      Department of Genetics, Eötvös Loránd University, Budapest, Hungary
    </li>
  </ul>
</div>

<a class="anchor" id="faq_what-does-a-protein-page-contain"></a>
<h3>
  What does a protein page contain?
</h3>
When you select a transcription factor or a target gene protein from the
Browsing table a protein page will be opened.

<div class="margin-left">
  <a class="anchor" id="faq_what-does-a-transcription-factor-protein-page-contain"></a>
  <h4>
    What does a transcription factor protein page contain?
  </h4>
  Besides basic information and a link to the
  <i>Uniprot</i>
  page of the protein there is a table that contains information about the
  target genes of the transcription factor.
  <br />
  If there are known binding sites of the transcription factor you will find a
  table containing their genomic locations and a list containing their DNA
  sequences in fasta format below the table.
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-does-a-target-gene-protein-page-contain"></a>
  <h4>
    What does a target gene protein page contain?
  </h4>
  Besides basic information and a link to the
  <i>Uniprot</i>
  page of the protein there is a table that contains information about the
  transcription factors of the target gene.
  <br />
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-does-a-transcription-factor-and-target-gene-protein-page-contain"></a>
  <h4>
    What does a transcription factor and target gene protein page contain?
  </h4>
  Some proteins act like transcription factors and target genes simultaneously.
  In some cases a transcription factor can regulate its own gene too.
  <br />
  Besides basic information and a link to the
  <i>Uniprot</i>
  page of the protein there is a table that contains information about the
  target genes of the protein.
  <br />
  There is also a table that contains information about the transcription
  factors of the protein.
  <br />
  If there are known binding sites of the transcription factor you will find a
  table containing their genomic locations and a list containing their DNA
  sequences in <i>fasta</i> format below the table.
</div>

<a class="anchor" id="faq_what-are-the-downloadable-data-formats"></a>
<h3>
  What are the downloadable data formats?
</h3>

<div class="margin-left">
  <a class="anchor" id="faq_what-do-interaction-tables-contain"></a>
  <h4>
    What do interaction tables contain?
  </h4>
  <p>
    Interaction tables are comma separated tables (<i>csv</i>) of transcription
    factor - target gene interactions that contain either interactions validated
    by small-scale experiments (<i>small-scale interaction table</i>) or
    large-scale experiments (<i>large-scale interaction table</i>), along with
    the following data: <i>Uniprot IDs</i>, protein names,
    <i>NCBI Gene IDs</i> of the transcription factors and the target genes; the
    names of the detection methods; the <i>Pubmed IDs</i> of the original
    publications; the name of the organism and the source database.
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-do-mitab"></a>
  <h4>What do <i>MITAB</i> tables contain?</h4>
  <p>
    Interaction MITABs contain transcription factor - target gene interactions
    in HUPO-PSI MITAB 2.8 format.
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-is-mitab"></a>
  <h4>What is the <i>MITAB</i> format?</h4>
  <p>
    <i>MITAB 2.8</i> is a standardized format introduced by the
    <i>Human Proteome Organization</i> (<i>HUPO</i>) which captures binary
    molecular interactions using standardized content and vocabulary. The
    <i>MITAB</i> files are tab-delimited tables containing 46 columns. A header
    for the <i>MITAB</i> tables is available
    <a href="/data/mitab_header_2.tsv">here</a>. <br />
    For more information on the <i>HUPO-PSI</i>'s molecular interaction format
    see: <a href="https://psicquic.github.io/MITAB28Format.html">link</a>
    <br />
    Sivade Dumousseau M et al. (2019) Encompassing new use cases - level 3.0 of
    the <i>HUPO-PSI</i> format for molecular interactions.
    <i>BMC Bioinformatics</i> <strong>19</strong>(1):134.
    <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2118-1">doi:
      10.1186/s12859-018-2118-1</a>.
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-do-binding-site-tables-contain"></a>
  <h4>
    What do binding site tables contain?
  </h4>
  <p>
    Binding site tables are comma separated tables (<i>csv</i>) of binding site
    annotations that contain the unique internal <i>TFLink IDs</i> of binding
    sites; <i>Uniprot IDs</i> and protein names of the transcription factors;
    the names of the detection method; <i>Pubmed IDs</i> of the original
    publications; the organism and the source database; the version of the
    genome assembly; the name of the chromosome; the start and end coordinates
    of site; and the strand.
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_what-do-binding-site-sequence-files-contain"></a>
  <h4>
    What do binding site sequence files contain?
  </h4>
  <p>
    Binding site sequence files are <i>fasta</i> files containing the DNA
    sequences of the transcription factor binding sites. The header of each
    sequence contains the unique internal <i>TFLink ID</i> of the binding site;
    the <i>Uniprot ID</i> and protein name of the transcription factor, the
    version of the genome assembly; the name of the chromosome; and the the
    start and end coordinates of site.
  </p>
</div>

<a class="anchor" id="faq_what-source-databases"></a>
<h3>
  What source databases provided the data of TFLink?
</h3>
<u>We use the following abbreviations in the table:</u>
<br />
Data based on small-scale experiments: SS
<br />
Data based on large-scale experiments: LS
<br />
<i>Homo sapiens</i>: Hs
<br />
<i>Mus musculus</i>: Mm
<br />
<i>Rattus norvegicus</i>: Rn
<br />
<i>Danio rerio</i>: Dr
<br />
<i>Drosophila melanogaster</i>: Dm
<br />
<i>Caenorhabditis elegans</i>: Ce
<br />
<i>Saccharomyces cerevisiae</i>: Sc
<br />

<table class="faq-table uk-table uk-table-striped uk-table-divider uk-table-small">
  <thead>
    <tr>
      <td colspan="3"></td>
      <td colspan="3">Nr. of data downloaded</td>
    </tr>
    <tr>
      <td></td>
      <td>Version</td>
      <td>Downloading date</td>
      <td>Type of data</td>
      <td>Nr. of integrated data</td>
      <td>Species</td>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td rowspan="2"><a href="http://gtrd.biouml.org/">GTRD</a></td>
      <td rowspan="2">18.06</td>
      <td rowspan="2">29/08/2018</td>
      <td>LS interactions</td>
      <td>5,341,065</td>
      <td rowspan="2">Hs, Mm, Rn, Dr, Ce, Sc</td>
    </tr>
    <tr>
      <td>LS binding sites</td>
      <td>5,341,065</td>
    </tr>
    <tr>
      <td rowspan="2">HTRIdb</td>
      <td rowspan="2">no longer available</td>
      <td rowspan="2">29/04/2017</td>
      <td>SS interactions</td>
      <td>1793</td>
      <td rowspan="2">Hs</td>
    </tr>
    <tr>
      <td>LS interactions</td>
      <td>47,565</td>
    </tr>
    <tr>
      <td rowspan="2"><a href="http://jaspar.genereg.net/">JASPAR</a></td>
      <td rowspan="2">2016</td>
      <td rowspan="2">16/08/2017</td>
      <td>SS binding sites</td>
      <td>7761</td>
      <td rowspan="2">Hs, Mm, Rn, Dm, Ce</td>
    </tr>
    <tr>
      <td>LS binding sites</td>
      <td>1,126,59</td>
    </tr>
    <tr>
      <td rowspan="4"><a href="http://www.oreganno.org/">ORegAnno</a></td>
      <td rowspan="4">2016 (the website is out of order)</td>
      <td rowspan="4">24/05/2017</td>
      <td>SS interactions</td>
      <td>1,762</td>
      <td rowspan="4">Hs, Mm, Rn, Dm, Ce, Sc</td>
    </tr>
    <tr>
      <td>LS interactions</td>
      <td>197,630</td>
    </tr>
    <tr>
      <td>SS binding sites</td>
      <td>47,409</td>
    </tr>
    <tr>
      <td>LS binding sites</td>
      <td>705,275</td>
    </tr>
    <tr>
      <td rowspan="2">
        <a href="http://redfly.ccr.buffalo.edu/">REDfly</a>
      </td>
      <td rowspan="2">5.5</td>
      <td rowspan="2">03/09/2018</td>
      <td>SS interactions</td>
      <td>1,100</td>
      <td rowspan="2">Dm</td>
    </tr>
    <tr>
      <td>SS binding sites</td>
      <td>1,787</td>
    </tr>
    <tr>
      <td rowspan="2">
        <a href="http://tagc.univ-mrs.fr/remap/index.php?page=about">ReMap</a>
      </td>
      <td rowspan="2">2018 (v1.2)</td>
      <td rowspan="2">16/07/2018</td>
      <td>LS interactions</td>
      <td>3,011,117</td>
      <td rowspan="2">Hs</td>
    </tr>
    <tr>
      <td>LS binding sites</td>
      <td>3,011,117</td>
    </tr>
    <tr>
      <td>
        <a href="http://rulai.cshl.edu/cgi-bin/TRED/tred.cgi?process=home">TRED</a>
      </td>
      <td>obtained from RegNetwork 2017</td>
      <td>06/08/2018</td>
      <td>SS interactions</td>
      <td>8,938</td>
      <td>Hs, Mm</td>
    </tr>
    <tr>
      <td><a href="https://www.grnpedia.org/trrust/">TRRUST</a></td>
      <td>2</td>
      <td>30/07/2018</td>
      <td>SS interactions</td>
      <td>16,188</td>
      <td>Hs, Mm</td>
    </tr>
    <tr>
      <td><a href="http://www.yeastract.com/">Yeastract</a></td>
      <td>-</td>
      <td>29/06/2017</td>
      <td>SS interactions</td>
      <td>159,625</td>
      <td>Sc</td>
    </tr>
  </tbody>
</table>

<a class="anchor" id="faq_where-can-I-find-the-the-interaction"></a>
<h3>
  Where can I find the the interaction and binding site data?
</h3>

<div class="margin-left">
  <a class="anchor" id="faq_where-can-i-find-the-interactions-small-scale"></a>
  <h4>
    Where can I find the interactions and binding sites based on small-scale
    experiments?
  </h4>
  <p>
    For data of small-scale experiments we provide options for browsing,
    searching and downloading data in different standardized formats
    (<i>MITAB</i>,
    <i>csv</i>, <i>gmt</i> and <i>fasta</i> files that include
    <i>Uniprot IDs</i>, protein names, <i>PubMed IDs</i>, etc.).
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_where-can-i-find-the-interactions-large-scale"></a>
  <h4>
    Where can I find the interactions and binding sites based on large-scale
    experiments?
  </h4>
  <p>
    Because of their quantity, the interactions and binding sites based on
    large-scale experiments are only available in forms of downloadable files,
    therefore cannot be browsed on the <i>TFLink</i> website. The downloadable
    files are accessible in standard formats: <i>MITAB</i>, <i>csv</i>,
    <i>gmt</i> and <i>fasta</i> files that include <i>Uniprot IDs</i>, protein
    names, <i>PubMed IDs</i>, etc.
  </p>
</div>

<div class="margin-left">
  <a class="anchor" id="faq_where-can-i-find-the-data-of-danio-rerio"></a>
  <h4>Where can I find the data of <i>Danio rerio</i>?</h4>
  <p>
    From <i>Danio rerio</i> only large-scale data was available in the source
    databases, therefore you can find these interaction and binding site data
    under the <i>Downloads</i> menu.
  </p>
</div>

<a class="anchor" id="faq_is-tflink-freely-available"></a>
<h3>Is <i>TFLink</i> freely available?</h3>
<p><i>TFLink</i> is freely available for non-commercial use.</p>

<script>
  $('#nav_faq').addClass('navbar-active');
</script>