{{ define "main" }}

{{ $statistics_file := "/static/data/statistics.csv" }}
{{ $sep := ";" }}
{{ $statistics := getCSV $sep $statistics_file }}

<div class="uk-cover-container uk-height-medium">
  <img src="/images/nyitokep_logoval_4.png" alt="" uk-cover />
</div>

<main aria-role="main">
  <div class="">
    <section>
      <div class="uk-grid-small" uk-grid>
        <div class="uk-card uk-card-body uk-width-3-5@m">
          <h1 class="">About</h1>
          <p style="text-align: justify;">
            <i>TFLink</i> gateway uniquely provides comprehensive and highly
            accurate information on
            transcription factor - target gene interactions, nucleotide
            sequences and genomic locations of transcription factor binding
            sites for <em>human</em> and six model organisms: <em>mouse</em> (<i>Mus musculus</i>),
            <em>rat</em> (<i>Rattus norvegicus</i>), <em>zebrafish</em> (<i>Danio rerio</i>),
            <em>fruit fly</em> (<i>Drosophila melanogaster</i>), <em>nematode</em>
            (<i>Caenorhabditis elegans</i>),
            and <em>yeast</em> (<i>Saccharomyces cerevisiae</i>). <i>TFLink</i> contains
            clearly identified data, and provides information about the sources:
            databases, experimental methods and publications. To create
            <i>TFLink</i>, we examined the large transcription
            factor databases, and selected ten resources for integration:
            <a href="https://saezlab.github.io/dorothea/" target="_blank">DoRothEA</a>,
            <a href="http://gtrd.biouml.org/" target="_blank">GTRD</a>,
            <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-405"
              target="_blank">HTRIdb</a>,
            <a href="http://jaspar.genereg.net/" target="_blank">JASPAR</a>,
            <a href="http://www.oreganno.org/" target="_blank">ORegAnno</a>,
            <a href="http://redfly.ccr.buffalo.edu/" target="_blank">REDfly</a>,
            <a href="http://remap.univ-amu.fr/" target="_blank">ReMap</a>,
            <a href="http://rulai.cshl.edu/cgi-bin/TRED/tred.cgi?process=home"
              target="_blank">TRED</a>,
            <a href="https://www.grnpedia.org/trrust/" target="_blank">TRRUST</a>, and
            <a href="http://www.yeastract.com/" target="_blank">Yeastract</a>.
            By exploiting these database sources, we integrated accurate, small-scale experimental
            data and the results
            of large-scale experiments (see the
            <a href="/faq/" target="_blank">FAQ</a>
            for the detailed list of methods).
          </p>
          <p>
            At the TFLink gateway you can <a href="/browse/" target="_blank">Browse</a> and search
            within the dataset, then open the <strong>Entry page</strong> of the selected
            <i>transcription factor</i> or <i>target gene</i>. Each entry page contains basic
            information about the
            protein or gene,
            its target genes and / or its transcription factors, binding site sequences, and
            crosslinks to proteins
            or genes at TFLink and to external databases and websites. You can download the
            interaction tables and
            binding site sequences of a certain protein or gene from the entry page, or download the
            content of the
            whole TFLink database at the <a href="/download/" target="_blank">Download</a> page.
          </p>
          <h3>Publication</h3>
          <p>
            Liska O, Bohár B, Hidas A, Korcsmáros T, Papp B, Fazekas D, Ari E (2022) 
            <a href="https://doi.org/10.1093/database/baac083" 
            target="_blank">TFLink: An integrated gateway to access transcription factor - target 
            gene interactions for multiple species</a>. <i>Database</i>, baac083
          </p>
        </div>
        <!-- <div class="uk-width-1-5@m"></div> -->
        <div class="uk-card uk-card-body uk-width-expand uk-width-1-5@m">
          <h1 class="">Contact us</h1>
          <p>
            TFLink gateway was created by Orsolya Liska<sup>1,2</sup>, Balázs Bohár<sup>2,3</sup>,
            András
            Hidas<sup>2,4</sup>, Tamás Korcsmáros<sup>3,5,6</sup>,
            Balázs Papp<sup>1</sup>, Dávid Fazekas<sup>2,3</sup> & Eszter Ari<sup>1,2</sup>
          </p>
          <ol class="author-list">
            <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html"
              target="_blank">
              <li>Synthetic and Systems Biology Unit, Institute of Biochemistry, Biological Research
                Centre, ELKH, Szeged,
                Hungary</li>
            </a>
            <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148" target="_blank">
              <li>Department of Genetics, ELTE Eötvös Loránd University, Budapest, Hungary</li>
            </a>
            <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
              <li>Earlham Institute, Norwich, UK</li>
            </a>
            <a href="https://www.voi.ecolres.hu/en/Andras.HIDAS" target="_blank">
              <li>Institute of Aquatic Ecology, Centre for Ecological Research, ELKH, Budapest,
                Hungary</li>
            </a>
            <a href="https://quadram.ac.uk" target="_blank">
              <li>Quadram Institute Bioscience, Norwich, UK</li>
            </a>
            <a href="https://www.imperial.ac.uk" target="_blank">
              <li>Imperial College London, Faculty of Medicine, London, UK</li>
            </a>
          </ol>
          <p>
            Please let us know if you have any questions or wish to integrate your data
            to TFLink:
          </p>
          <p>
            <a href="mailto:tflink.net@gmail.com" target="_top">tflink.net@gmail.com</a>
          </p>
        </div>
      </div>
    </section>
    <hr />

    <section class="uk-container">
      <div class="uk-grid-small" uk-grid>
        <div class="uk-card uk-card-body">
          <h2 class="">Summary statistics of all small- and large-scale data</h2>
          <table class="uk-table uk-table-justify statistics">
            <thead>
              <tr>
                <th class="uk-table-expand">Organism</th>
                <th class=""></th>
                <th class="uk-table-expand">Scale</th>
                <th class="">Nr. of TFs⁴</th>
                <th class="">Nr. of target genes</th>
                <th class="">Nr. of interactions</th>
                <th class="">Nr. of binding sites⁵</th>
                <th class="">Nr. of binding sequences</th>
              </tr>
            </thead>
            <tbody>
              {{ range $i, $row:= (after 1 $statistics) }}
              <tr>
                {{ $rowspan:= 1 }}
                {{ if eq (mod $i 3) 0 }}
                {{ $rowspan = 3 }}
                <td class="stat-organism" style="vertical-align: middle" rowspan=3>
                  {{ index $row 0 }}</td>
                {{ end }}
                {{ if (and (lt $i 21) (eq (mod $i 3) 0))}}
                <td rowspan={{$rowspan}} class="siluette">
                  <img
                    src="{{ printf `/images/siluettes/%s.svg` (replace (index $row 0) ` ` `_`) }}" />
                </td>
                {{ end }}
                {{ if gt $i 20 }}
                <td></td>
                {{ end }}
                <td align="center">{{ index $row 1 }}</td>
                {{ range after 2 $row }}
                <td align="right">{{ lang.NumFmt 0 (int (index .)) "- . ," }}</td>
                {{ end }}
              </tr>
              {{ end }}
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <hr />
    <div class="uk-container">
      ⁴ TF: transcription factor<br />
      ⁵ In some cases the number of binding sites and the number of binding sequences are different
      because among the
      data we
      downloaded from JASPAR there are binding sequences with missing localization, for example when
      random sequences
      were
      investigated with the SELEX.<br />
      ⁶ The total number of TFs, target genes, etc. can be different from the sum of small- and
      large-scale data due to
      overlapping items.

    </div>
  </div>
</main>
{{ end }} {{ define "footer"}}
<div class="uk-grid-medium uk-flex-center uk-flex-middle uk-child-width-1-6" uk-grid>
  <div class="logo">
    <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-index.html" target="_blank">
      <img src="/images/brc_logo.png" width="60%" />
    </a>
  </div>
  <div class="logo">
    <a href="https://www.earlham.ac.uk/korcsmaros-group" target="_blank">
      <img src="/images/EI-double-large-trans-highquality.png" />
    </a>
  </div>
  <div class="logo">
    <a href="https://www.hcemm.eu/teams/metabolic-and-cardiovascular-diseases/hcemm-brc-metabolic-systems-biology-research-group/"
      target="_blank">
      <img src="/images/H-CEMM_Logo_RGB.jpg" />
    </a>
  </div>
  <div class="logo">
    <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148" target="_blank">
      <img src="/images/elte_cimer_szines.png" width="70%" />
    </a>
  </div>
  <div class="logo">
    <a href="https://www.imperial.ac.uk" target="_blank">
      <img src="/images/imperial-college.png" />
    </a>
  </div>
  <div class="logo">
    <a href="https://github.com/korcsmarosgroup/TFlink" target="_blank">
      <img src="/images/Octocat.png" width="70%" />
    </a>
  </div>
</div>
{{ end }}