{{ define "main" }}

<section class="uk-container uk-container-medium">
  <span class="title">
    <h1>Download</h1>
  </span>
  <hr />
  <div class="uk-grid">
    <div id="species_select_div" class="uk-width-1-2@s"></div>
    <div id="species_siluette" class="uk-width-1-2@s siluette-large">
    </div>

  </div>
  <p></p>
  <div class="uk-grid" data-uk-grid-margin>
    <div id="download_div_ss" class="uk-width-1-3@s"></div>
    <div id="download_div_ls" class="uk-width-1-3@s"></div>
    <div id="download_div_all" class="uk-width-1-3@s"></div>
  </div>

  <p>&nbsp;</p>

  <div>
    <h3>Description</h3>
    <ul>
      <li>
        <strong>Interaction table</strong> files are comma separated tables
        <i>(csv)</i> of transcription factor - target gene interactions that
        contain either interactions validated by small-scale experiments
        or large-scale experiments or these two data altogether. 
        All tables contain the following data: <i>Uniprot IDs</i>,
        gene names, <i>NCBI Gene IDs</i> of transcription factors and
        target genes, names of the detection methods,
        <i>Pubmed IDs</i>
        of the original publications, name of the organism, source
        databases, and indication about if the data were confirmed by 
        small-scale evidences.
      </li>
      <li>
        <strong>Interaction MITAB</strong> files contain transcription factor -
        target gene interactions in <i>HUPO-PSI MITAB 2.8</i> format. 
        The detailed description of the format and a header for <i>MITAB</i> 
        tables are available in the
        <a href="https://tflink.net/faq/" target="_blank">FAQ</a>.
      </li>
      <li>
        <strong>Interaction GMT</strong> (Gene Matrix Transposed) is a tab 
        delimited file format that describes gene sets – target genes of a 
        transcription factor – in each row. The first and second column 
        contains information about the transcription factors (various IDs and 
        gene names). The first cell in each row is always unique. From the 
        third to the last column the target genes of the transcription factor 
        are listed. The number of target genes can vary from transcription 
        factor to transcription factor, therefore the number of cells can be 
        different in every row. The user can choose between <i>GMT</i> files 
        with <i>Uniprot IDs</i>, <i>NCBI Gene IDs</i>, and <i>gene names</i>.
      </li>
      <li>
        <strong>Binding site</strong> table files are comma separated tables 
        (<i>csv</i>) of binding site annotations that contain unique 
        <i>TFLink IDs</i> of binding sites, <i>Uniprot IDs</i> and gene names 
        of the transcription factors, the names of the detection method, 
        indication if it is a small- or a large-scale method, <i>Pubmed IDs</i> 
        of the original publications, the organism, the source database, the 
        version of the genome assembly, the name of the chromosome, the start 
        and end coordinates of the sites, and the coding strand.
      </li>
      <li>
        <strong>Binding site sequence</strong> files are <i>fasta</i> files 
        containing the DNA sequences of the transcription factor binding sites. 
        The header of each sequence contains the unique internal 
        <i>TFLink ID</i> of the binding site, the <i>Uniprot ID</i> and gene 
        name of the transcription factor, the version of the genome assembly, 
        the name of the chromosome, and the start and end coordinates of the 
        sites.
      </li>
    </ul>
  </div>
</section>

<script>
  $('#nav_download').addClass('navbar-active');
</script>

{{ $script := .Site.Data.webpack.download }} {{ with $script.js }}
<script src="{{ relURL . }}"></script>
{{ end }} {{ end }}
